<!DOCTYPE html> <html lang="en" class=> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> </title> <meta name="author" content=" "/> <meta name="description" content="The Vallejos Research Group Website "/> <meta name="keywords" content="Bayesian Single-cell RNA-seq EHR"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/favicon.png"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://vallejosgroup.github.io/"> <link rel="apple-touch-icon" href="https://vallejosgroup.github.io/assets/img/favicon.png"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <div class="font-sans"> <header> <nav id="navbar" class=" navbar font-sans font-weight-lighter bg-opacity-30 bg-slate-900 backdrop-filter border-b-2 border-gray-400 border-transparent backdrop-blur-sm navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="https://vallejosgroup.github.io/"><picture> <source srcset="/assets/img/logo-480.webp" type="image/webp"> <source srcset="/assets/img/logo.png" type="image/png"> <img src="/assets/img/logo" alt="" width="100" height="56"> </source></source></picture></a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link navbarcolor" href="/"> About<span class="sr-only"> (current)</span></a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/people/">People</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/publications/">Publications</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/software/">Software</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/contact/">Contact Us</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <article> <div class="relative rounded-xl overflow-hidden"> <picture> <source srcset="https://vallejosgroup.github.io/assets/img/c.-vallejos-group-1400.webp" type="image/webp"> <source srcset="https://vallejosgroup.github.io/assets/img/c.-vallejos-group.jpg" type="image/jpeg"> <img src="https://vallejosgroup.github.io/assets/img/c.-vallejos-group-1400.webp" alt="Avatar" class="object-cover h-30 w-100 rounded-xl"> </source></source></picture> <div class="absolute w-full py-2.5 bottom-0 inset-x-0 border-slate-400 bg-slate-400 bg-opacity-40 rounded-b-xl text-center leading-4 backdrop-filter backdrop-blur-sm">
<h1 class="post-title"> <span class="font-extralight text-2xl">Vallejos Group</span> </h1> <p class="desc text-lg">Understanding heterogeneity in complex biomedical data</p> </div> </div> <br> <div class="publications"> <h1 class="text-2xl font-extralight">About</h1> <p>While biomedical data sometimes classifies as “big data” (where the number of samples and/or variables is large), complexity is its most prominent feature. This arises from a combination of different sources of heterogeneity: heterogeneity across individuals in a population (e.g. response to treatment), heterogeneity in terms of the type of data we collect (e.g. health records &amp; genomics) and heterogeneity that is introduced by the data collection process (e.g. measurement error).</p> <p>We focus on the development of novel statistical methodology to address and study these sources of heterogeneity. This is a highly multidisciplinary task: from the understanding of complex biomedical problems and technologies, to the development of new methodology and the implementation of open-source analysis tools. Our current research focuses on two areas of application. Firstly, single-cell RNA-sequencing, a cutting-edge experimental technique that allows genome-wide quantification of gene expression on a cell-by-cell basis. Secondly, electronic health records research, to develop predictive models based on observational data that is routinely collected by health providers (e.g. NHS). Developing computational tools that can make full advantage of the rich information provided by these data sources is ought to improve our understanding of health and disease, playing an important role in precision medicine initiatives.</p> <div> <br> <div class="news"> <h1 class="text-2xl font-extralight">News</h1> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row">Jan 31, 2024</th> <td> <a class="news-title" href="/news/announcement_5/">Nathan has passed his viva!</a> </td> </tr> <tr> <th scope="row">May 26, 2022</th> <td> <a class="news-title" href="/news/announcement_4/">Cata got tenure after a successful ESAT review.</a> </td> </tr> <tr> <th scope="row">May 6, 2022</th> <td> <a class="news-title" href="/news/announcement_3/">Alan has passed his viva!</a> </td> </tr> <tr> <th scope="row">Oct 1, 2021</th> <td> <a class="news-title" href="/news/announcement_2/">Linda and Elena have joined the group!</a> </td> </tr> <tr> <th scope="row">Feb 10, 2021</th> <td> Andrew has been awarded an ISSF grant to perform a single-cell multiomics study of eye organoids </td> </tr> </table> </div> </div> <div class="publications"> <h1 class="text-2xl font-extralight">Selected Publications</h1> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Biomet</abbr></div> <div id="https://doi.org/10.1002/bimj.202300060" class="col-sm-8"> <div class="title">A review on statistical and machine learning competing risks methods</div> <div class="author"> <em>Karla Monterrubio-Gómez</em>,  <em>Nathan Constantine-Cooke</em>, and <em>Catalina A. Vallejos</em> </div> <div class="periodical"> <em>Biometrical Journal</em> Feb 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.202300060" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Abstract When modeling competing risks (CR) survival data, several techniques have been proposed in both the statistical and machine learning literature. State-of-the-art methods have extended classical approaches with more flexible assumptions that can improve predictive performance, allow high-dimensional data and missing values, among others. Despite this, modern approaches have not been widely employed in applied settings. This article aims to aid the uptake of such methods by providing a condensed compendium of CR survival methods with a unified notation and interpretation across approaches. We highlight available software and, when possible, demonstrate their usage via reproducible R vignettes. Moreover, we discuss two major concerns that can affect benchmark studies in this context: the choice of performance metrics and reproducibility.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">https://doi.org/10.1002/bimj.202300060</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Biomet}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Monterrubio-Gómez, Karla and Constantine-Cooke, Nathan and Vallejos, Catalina A.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A review on statistical and machine learning competing risks methods}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biometrical Journal}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{66}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2300060}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{competing risks, risk prediction, survival analysis, time-to-event data}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/pdf/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CGH</abbr></div> <div id="Constantine-Cooke2023" class="col-sm-8"> <div class="title">Longitudinal Fecal Calprotectin Profiles Characterize Disease Course Heterogeneity in Crohn’s Disease</div> <div class="author"> <em>Nathan Constantine-Cooke</em>,  <em>Karla Monterrubio-Gómez</em>, Nikolas Plevris, Lauranne A.A.P. Derikx, Beatriz Gros, Gareth-Rhys Jones, Riccardo E. Marioni, <a href="https://charlielees.com" target="_blank" rel="noopener noreferrer">Charlie W. Lees</a>, and <em>Catalina A. Vallejos</em> </div> <div class="periodical"> <em>Clinical Gastroenterology and Hepatology</em> Oct 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/Constantine-Cooke2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Background and Aims: The progressive nature of Crohn’s disease is highly variable and hard to predict. In addition, symptoms correlate poorly with mucosal inflammation. There is therefore an urgent need to better characterize the heterogeneity of disease trajectories in Crohn’s disease by utilizing objective markers of inflammation. We aimed to better understand this heterogeneity by clustering Crohn’s disease patients with similar longitudinal fecal calprotectin profiles. Methods: We performed a retrospective cohort study at the Edinburgh IBD Unit, a tertiary referral center, and used latent class mixed models to cluster Crohn’s disease subjects using fecal calprotectin observed within 5 years of diagnosis. Information criteria, alluvial plots, and cluster trajectories were used to decide the optimal number of clusters. Chi-square test, Fisher’s exact test, and analysis of variance were used to test for associations with variables commonly assessed at diagnosis. Results: Our study cohort comprised 356 patients with newly diagnosed Crohn’s disease and 2856 fecal calprotectin measurements taken within 5 years of diagnosis (median 7 per subject). Four distinct clusters were identified by characteristic calprotectin profiles: a cluster with consistently high fecal calprotectin and 3 clusters characterized by different downward longitudinal trends. Cluster membership was significantly associated with smoking (P = .015), upper gastrointestinal involvement (P &lt; .001), and early biologic therapy (P &lt; .001). Conclusions: Our analysis demonstrates a novel approach to characterizing the heterogeneity of Crohn’s disease by using fecal calprotectin. The group profiles do not simply reflect different treatment regimens and do not mirror classical disease progression endpoints.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Constantine-Cooke2023</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{CGH}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{Constantine-Cooke2023.pdf}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Longitudinal Fecal Calprotectin Profiles Characterize Disease Course Heterogeneity in {{Crohn}}'s Disease}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{{Constantine-Cooke}, Nathan and {Monterrubio-G{\'o}mez}, Karla and Plevris, Nikolas and Derikx, Lauranne A.A.P. and Gros, Beatriz and Jones, Gareth-Rhys and Marioni, Riccardo E. and Lees, Charlie W. and Vallejos, Catalina A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Gastroenterology and Hepatology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2918-2927.e6}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{{Elsevier}}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1542-3565}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cgh.2023.03.026}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-05-24}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NatAge</abbr></div> <div id="Cheng_2023" class="col-sm-8"> <div class="title">Development and validation of DNA methylation scores in two European cohorts augment 10-year risk prediction of type 2 diabetes</div> <div class="author"> <em>Yipeng Cheng</em>, Danni A. Gadd, Christian Gieger,  <em>Karla Monterrubio-Gómez</em>, Yufei Zhang, Imrich Berta, Michael J. Stam, Natalia Szlachetka, Evgenii Lobzaev, Nicola Wrobel, Lee Murphy, Archie Campbell, Cliff Nangle, Rosie M. Walker, Chloe Fawns-Ritchie, Annette Peters, Wolfgang Rathmann, David J. Porteous, Kathryn L. Evans, Andrew M. McIntosh, Timothy I. Cannings, Melanie Waldenberger, Andrea Ganna, Daniel L. McCartney,  <em>Catalina A. Vallejos</em>, and Riccardo E. Marioni </div> <div class="periodical"> <em>Nature Aging</em> Apr 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038%2Fs43587-023-00391-4" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> <a href="/assets/pdf/cheng2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Type 2 diabetes mellitus (T2D) presents a major health and economic burden that could be alleviated with improved early prediction and intervention. While standard risk factors have shown good predictive performance, we show that the use of blood-based DNA methylation information leads to a significant improvement in the prediction of 10-year T2D incidence risk. Previous studies have been largely constrained by linear assumptions, the use of cytosine–guanine pairs one-at-a-time and binary outcomes. We present a flexible approach (via an R package, MethylPipeR) based on a range of linear and tree-ensemble models that incorporate time-to-event data for prediction. Using the Generation Scotland cohort (training set ncases = 374, ncontrols = 9,461; test set ncases = 252, ncontrols = 4,526) our best-performing model (area under the receiver operating characteristic curve (AUC) = 0.872, area under the precision-recall curve (PRAUC) = 0.302) showed notable improvement in 10-year onset prediction beyond standard risk factors (AUC = 0.839, precision-recall AUC = 0.227). Replication was observed in the German-based KORA study (n = 1,451, ncases = 142, P = 1.6x10-5).</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Cheng_2023</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s43587-023-00391-4}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038%2Fs43587-023-00391-4}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media {LLC}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{450--458}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Cheng, Yipeng and Gadd, Danni A. and Gieger, Christian and Monterrubio-G{\'{o}}mez, Karla and Zhang, Yufei and Berta, Imrich and Stam, Michael J. and Szlachetka, Natalia and Lobzaev, Evgenii and Wrobel, Nicola and Murphy, Lee and Campbell, Archie and Nangle, Cliff and Walker, Rosie M. and Fawns-Ritchie, Chloe and Peters, Annette and Rathmann, Wolfgang and Porteous, David J. and Evans, Kathryn L. and McIntosh, Andrew M. and Cannings, Timothy I. and Waldenberger, Melanie and Ganna, Andrea and McCartney, Daniel L. and Vallejos, Catalina A. and Marioni, Riccardo E.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development and validation of {DNA} methylation scores in two European cohorts augment 10-year risk prediction of type 2 diabetes}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Aging}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NatAge}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{cheng2021.pdf}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Kapourani2021" class="col-sm-8"> <div class="title">scMET: Bayesian modeling of DNA methylation heterogeneity at single-cell resolution</div> <div class="author"> <em>Chantriolnt-Andreas Kapourani</em>, Ricard Argelaguet, <a href="https://datascience.sissa.it/person/11/guido-sanguinetti" target="_blank" rel="noopener noreferrer">Guido Sanguinetti</a>, and <em>Catalina A Vallejos</em> </div> <div class="periodical"> <em>Genome Biology</em> Apr 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-021-02329-8" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>High-throughput single-cell measurements of DNA methylomes can quantify methylation heterogeneity and uncover its role in gene regulation. However, technical limitations and sparse coverage can preclude this task. scMET is a hierarchical Bayesian model which overcomes sparsity, sharing information across cells and genomic features to robustly quantify genuine biological heterogeneity. scMET can identify highly variable features that drive epigenetic heterogeneity, and perform differential methylation and variability analyses. We illustrate how scMET facilitates the characterization of epigenetically distinct cell populations and how it enables the formulation of novel hypotheses on the epigenetic regulation of gene expression. scMET is available at https://github.com/andreaskapou/scMET.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Kapourani2021</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kapourani, Chantriolnt-Andreas and Argelaguet, Ricard and Sanguinetti, Guido and Vallejos, Catalina A}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2021/04/20}</span><span class="p">,</span>
  <span class="na">date-added</span> <span class="p">=</span> <span class="s">{2022-02-26 12:43:46 +0000}</span><span class="p">,</span>
  <span class="na">date-modified</span> <span class="p">=</span> <span class="s">{2022-02-26 12:43:46 +0000}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Kapourani2021}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{114}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{scMET: Bayesian modeling of DNA methylation heterogeneity at single-cell resolution}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%63%61%74%61%6C%69%6E%61.%76%61%6C%6C%65%6A%6F%73@%69%67%6D%6D.%65%64.%61%63.%75%6B" title="email"><i class="fas fa-envelope"></i></a> <a href="https://github.com/VallejosGroup" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a> <a href="https://twitter.com/CataVallejosM" title="Twitter" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a> </div> <div class="contact-note"> </div> </div>  </div> </div> <footer class="fixed-bottom"> <div class="container mt-0" style="text-align: right;"> © Copyright 2024 </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script src="/assets/js/zoom.js"></script> <script src="/assets/js/common.js"></script> <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </article>
</div> </div>
</div></body> </html>
