<!DOCTYPE html> <html lang="en" class=> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> </title> <meta name="author" content=" "/> <meta name="description" content="The Vallejos Research Group Website "/> <meta name="keywords" content="Bayesian Single-cell RNA-seq EHR"/> <meta property="og:site_name" content=" "/> <meta property="og:type" content="website"/> <meta property="og:title" content=" | About"/> <meta property="og:url" content="https://vallejosgroup.github.io/"/> <meta property="og:description" content="The Vallejos Research Group Website "/> <meta property="og:image" content="/assets/img/open-graph-logo.png"/> <meta property="og:locale" content="en"/> <meta name="twitter:card" content="summary"/> <meta name="twitter:title" content="About"/> <meta name="twitter:description" content="The Vallejos Research Group Website "/> <meta name="twitter:image" content="/assets/img/open-graph-logo.png"/> <meta name="twitter:site" content="@CataVallejosM"/> <meta name="twitter:creator" content="@CataVallejosM"/> <script type="application/ld+json">
      {
        "author":
        {
          "@type": "Person",
          "name": "  "
        },
        "url": "https://vallejosgroup.github.io/",
        "@type": "WebSite",
        "description": "The Vallejos Research Group Website
",
        "headline": "About",
        "sameAs": ["https://orcid.org/0000-0003-3638-1960", "https://scholar.google.com/citations?user=lkdrwm0AAAAJ", "https://github.com/VallejosGroup", "https://www.linkedin.com/in/catalina-vallejos-61b41956", "https://twitter.com/CataVallejosM"],
        "name": "  ",
        "@context": "https://schema.org"
      }
    </script> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/favicon.png"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://vallejosgroup.github.io/"> <link rel="apple-touch-icon" href="https://vallejosgroup.github.io/assets/img/favicon.png"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <div class="font-sans"> <header> <nav id="navbar" class=" navbar font-sans font-weight-lighter bg-opacity-30 bg-slate-900 backdrop-filter border-b-2 border-gray-400 border-transparent backdrop-blur-sm navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="https://vallejosgroup.github.io/"><picture> <source srcset="/assets/img/logo-480.webp" type="image/webp"> <source srcset="/assets/img/logo.png" type="image/png"> <img src="/assets/img/logo" alt="" width="100" height="56"> </source></source></picture></a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link navbarcolor" href="/"> About<span class="sr-only"> (current)</span></a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/people/">People</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/publications/">Publications</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/software/">Software</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/contact/">Contact Us</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <article> <div class="relative rounded-xl overflow-hidden"> <picture> <source srcset="https://vallejosgroup.github.io/assets/img/c.-vallejos-group-1400.webp" type="image/webp"> <source srcset="https://vallejosgroup.github.io/assets/img/c.-vallejos-group.jpg" type="image/jpeg"> <img src="https://vallejosgroup.github.io/assets/img/c.-vallejos-group-1400.webp" alt="Avatar" class="object-cover h-30 w-100 rounded-xl"> </source></source></picture> <div class="absolute w-full py-2.5 bottom-0 inset-x-0 border-slate-400 bg-slate-400 bg-opacity-40 rounded-b-xl text-center leading-4 backdrop-filter backdrop-blur-sm">
<h1 class="post-title"> <span class="font-extralight text-2xl">Vallejos Group</span> </h1> <p class="desc text-lg">Understanding heterogeneity in complex biomedical data</p> </div> </div> <br> <div class="publications"> <h1 class="text-2xl font-extralight">About</h1> <p>While biomedical data sometimes classifies as “big data” (where the number of samples and/or variables is large), complexity is its most prominent feature. This arises from a combination of different sources of heterogeneity: heterogeneity across individuals in a population (e.g. response to treatment), heterogeneity in terms of the type of data we collect (e.g. health records &amp; genomics) and heterogeneity that is introduced by the data collection process (e.g. measurement error).</p> <p>We focus on the development of novel statistical methodology to address and study these sources of heterogeneity. This is a highly multidisciplinary task: from the understanding of complex biomedical problems and technologies, to the development of new methodology and the implementation of open-source analysis tools. Our current research focuses on two areas of application. Firstly, single-cell RNA-sequencing, a cutting-edge experimental technique that allows genome-wide quantification of gene expression on a cell-by-cell basis. Secondly, electronic health records research, to develop predictive models based on observational data that is routinely collected by health providers (e.g. NHS). Developing computational tools that can make full advantage of the rich information provided by these data sources is ought to improve our understanding of health and disease, playing an important role in precision medicine initiatives.</p> <div> <br> <div class="news"> <h1 class="text-2xl font-extralight">News</h1> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row">Jan 31, 2024</th> <td> <a class="news-title" href="/news/announcement_5/">Nathan has passed his viva!</a> </td> </tr> <tr> <th scope="row">May 26, 2022</th> <td> <a class="news-title" href="/news/announcement_4/">Cata got tenure after a successful ESAT review.</a> </td> </tr> <tr> <th scope="row">May 6, 2022</th> <td> <a class="news-title" href="/news/announcement_3/">Alan has passed his viva!</a> </td> </tr> <tr> <th scope="row">Oct 1, 2021</th> <td> <a class="news-title" href="/news/announcement_2/">Linda and Elena have joined the group!</a> </td> </tr> </table> </div> </div> <div class="publications"> <h1 class="text-2xl font-extralight">Selected Publications</h1> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">medRxiv</abbr></div> <div id="Constantine-Cooke2024.11.08.24316916" class="col-sm-8"> <div class="title">Large-scale clustering of longitudinal faecal calprotectin and C-reactive protein profiles in inflammatory bowel disease</div> <div class="author"> <em>Nathan Constantine-Cooke</em>, Marie Vibeke Vestergaard, Nikolas Plevris,  <em>Karla Monterrubio-Gómez</em>, Clara Ramos Belinchón, Solomon Ong, Alexander T. Elford, Beatriz Gros, Aleksejs Sazonovs, Gareth-Rhys Jones, Tine Jess,  <em>Catalina A. Vallejos</em>, and <a href="https://charlielees.com" target="_blank" rel="noopener noreferrer">Charlie W. Lees</a> </div> <div class="periodical"> <em>medRxiv</em> Jan 2025 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.medrxiv.org/content/early/2025/01/17/2024.11.08.24316916" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background Crohn’s disease (CD) and ulcerative colitis (UC) are highly heterogeneous, dynamic and unpredictable, with a marked disconnect between symptoms and intestinal inflammation. Attempts to classify inflammatory bowel disease (IBD) subphenotypes to inform clinical decision making have been limited. We aimed to describe the latent disease heterogeneity by modelling routinely collected faecal calprotectin (FC) and CRP data, describing dynamic longitudinal inflammatory patterns in IBD.Methods In this longitudinal study, we analysed patient-level post-diagnosis measurements of FC and CRP in two European cohorts. Latent class mixed models were used to cluster individuals with similar longitudinal profiles. Associations between cluster assignment and baseline characteristics were quantified using multinomial logistic regression. Differences in advanced therapy use across clusters were also explored. Finally, we considered uncertainty in cluster assignments with respect to follow-up length and the overlap between FC and CRP clusters. We included 1036 patients in the FC discovery analysis (Lothian) with a total of 10545 FC observations (median 9 per subject, IQR 6–13), and 7880 patients in the replication (Denmark). The CRP discovery analysis consisted of 1838 patients with 49364 measurements (median 20 per subject; IQR 10–36), with 10041 patients in the replication cohort.Findings Eight distinct clusters of inflammatory behaviour over time were identified in the FC and CRP analysis for the Scottish cohort. This model was then applied to the Danish replication cohort, with similar patterns observed in both the Scottish and Danish populations. The clusters, FC1–8 and CRP1–8, were ordered from the lowest cumulative inflammatory burden to the highest. The clusters included groups with high diagnostic levels of inflammation which rapidly normalised, groups where high inflammation levels persisted throughout the full seven years of observation, and a series of intermediates including delayed remitters and relapsing remitters. CD and UC patients were unevenly distributed across the clusters. In UC, male sex was associated with the poorest prognostic cluster (FC8). The use and timing of advanced therapy was associated with cluster assignment, with the highest use of early advanced therapy in FC1. Of note, FC8 and CRP8 captured consistently high patterns of inflammation despite a high proportion of patients receiving advanced therapy, particularly for CD individuals. We observed that uncertainty in cluster assignments was higher for individuals with short longitudinal follow-up, particularly between clusters capturing similar earlier inflammation patterns. There was broadly poor agreement between FC and CRP clusters in keeping with the need to monitor both in clinical practice.Interpretation Distinct patterns of inflammatory behaviour over time are evident in patients with IBD. Cluster assignment is associated with disease type and both the use and timing of advanced therapy. These data pave the way for a deeper understanding of disease heterogeneity in IBD and enhanced patient stratification in the clinic.Competing Interest StatementNP has served as a speaker for Janssen, Takeda and Pfizer. BG has acted as consultant to Galapagos and Abbvie and as speaker for Abbvie, Jansen, Takeda, Pfizer and Galapagos. GRJ has served as a speaker for Takeda, Janssen, Abbvie, Fresnius and Ferring. TJ has served as a speaker and/or consultant for Ferring and Pfizer. CWL has acted as a speaker and/or consultant to AbbVie, Janssen, Takeda, Pfizer, Galapagos, GSK, Gilead, Vifor Pharma, Ferring, Dr Falk, BMS, Boehringer Ingelheim, Eli Lilly, Merck, Novartis, Sandoz, Celltrion, Cellgene, Amgen, Samsung Bioepis, Fresenius Kabi, Tillotts, Kuma Health, Trellus Health and Iterative Health. None of the other authors report any conflicts of interest.Funding StatementCWL is funded by a UKRI (UK Research and Innovation) Future Leaders Fellowship ’Predicting outcomes in IBD’ (MR/S034919/1). G-RJ is funded by a Wellcome Trust Clinical Research Career Development Fellowship. NC-C was partially supported by the Medical Research Council and The University of Edinburgh via a Precision Medicine PhD studentship (MR/N013166/1). MVV, AS, and TJ are funded by a Center of Excellence Grant (DNRF148) to TJ from the Danish National Research Foundation.Author DeclarationsI confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.YesThe details of the IRB/oversight body that provided approval or exemption for the research described are given below:Usage of the Scottish dataset was approved by the local Caldicott Guardian (Project ID: CRD18002, registered NHS Lothian information asset #IAR-954). In Denmark, studies based on registry data alone are not required to obtain permission from the regional ethics committees as confirmed by The Central Denmark Region Committees on Health Research Ethics (legislation: 1-10-72-148-19)I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.YesI understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).YesI have followed all appropriate research reporting guidelines, such as any relevant EQUATOR Network research reporting checklist(s) and other pertinent material, if applicable.YesAs the data collected for this study has been derived from unconsented patient data, it is not possible to share subject-level data with external entities. Detailed summary level data is available online at https://vallejosgroup.github.io/Lothian-IBDR. The code used to conduct the analysis is also publicly available (https://github.com/VallejosGroup/Lothian-IBDR). https://vallejosgroup.github.io/IBD-Inflammatory-Patterns/ https://github.com/VallejosGroup/IBD-Inflammatory-Patterns</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Constantine-Cooke2024.11.08.24316916</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Constantine-Cooke, Nathan and Vestergaard, Marie Vibeke and Plevris, Nikolas and Monterrubio-G{\'o}mez, Karla and Ramos Belinch{\'o}n, Clara and Ong, Solomon and Elford, Alexander T. and Gros, Beatriz and Sazonovs, Aleksejs and Jones, Gareth-Rhys and Jess, Tine and Vallejos, Catalina A. and Lees, Charlie W.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Large-scale clustering of longitudinal faecal calprotectin and C-reactive protein profiles in inflammatory bowel disease}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2024.11.08.24316916}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2024.11.08.24316916}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory Press}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2025/01/17/2024.11.08.24316916}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2025/01/17/2024.11.08.24316916.full.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin Epigen</abbr></div> <div id="Bernabeu_2025" class="col-sm-8"> <div class="title">Blood-based epigenome-wide association study and prediction of alcohol consumption</div> <div class="author"> <em>Elena Bernabeu</em>, Aleksandra D. Chybowska, Jacob K. Kresovich, Matthew Suderman, Daniel L. McCartney, Robert F. Hillary, Janie Corley, Maria Del C. Valdés-Hernández, Susana Muñoz Maniega, Mark E. Bastin, Joanna M. Wardlaw, Zongli Xu, Dale P. Sandler, Archie Campbell, Sarah E. Harris, Andrew M. McIntosh, Jack A. Taylor, Paul Yousefi, Simon R. Cox, Kathryn L. Evans, Matthew R. Robinson,  <em>Catalina A. Vallejos</em>, and Riccardo E. Marioni </div> <div class="periodical"> <em>Clinical Epigenetics</em> Jan 2025 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1186/s13148-025-01818-y" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Alcohol consumption is an important risk factor for multiple diseases. It is typically assessed via self-report, which is open to measurement error through recall bias. Instead, molecular data such as blood-based DNA methylation (DNAm) could be used to derive a more objective measure of alcohol consumption by incorporating information from cytosine-phosphate-guanine (CpG) sites known to be linked to the trait. Here, we explore the epigenetic architecture of self-reported weekly units of alcohol consumption in the Generation Scotland study. We first create a blood-based epigenetic score (EpiScore) of alcohol consumption using elastic net penalized linear regression. We explore the effect of pre-filtering for CpG features ahead of elastic net, as well as differential patterns by sex and by units consumed in the last week relative to an average week. The final EpiScore was trained on 16,717 individuals and tested in four external cohorts: the Lothian Birth Cohorts (LBC) of 1921 and 1936, the Sister Study, and the Avon Longitudinal Study of Parents and Children (total N across studies &gt; 10,000). The maximum Pearson correlation between the EpiScore and self-reported alcohol consumption within cohort ranged from 0.41 to 0.53. In LBC1936, higher EpiScore levels had significant associations with poorer global brain imaging metrics, whereas self-reported alcohol consumption did not. Finally, we identified two novel CpG loci via a Bayesian penalized regression epigenome-wide association study of alcohol consumption. Together, these findings show how DNAm can objectively characterize patterns of alcohol consumption that associate with brain health, unlike self-reported estimates.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Bernabeu_2025</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Clin Epigen}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Blood-based epigenome-wide association study and prediction of alcohol consumption}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1868-7083}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1186/s13148-025-01818-y}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13148-025-01818-y}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Epigenetics}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media LLC}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Bernabeu, Elena and Chybowska, Aleksandra D. and Kresovich, Jacob K. and Suderman, Matthew and McCartney, Daniel L. and Hillary, Robert F. and Corley, Janie and Valdés-Hernández, Maria Del C. and Maniega, Susana Muñoz and Bastin, Mark E. and Wardlaw, Joanna M. and Xu, Zongli and Sandler, Dale P. and Campbell, Archie and Harris, Sarah E. and McIntosh, Andrew M. and Taylor, Jack A. and Yousefi, Paul and Cox, Simon R. and Evans, Kathryn L. and Robinson, Matthew R. and Vallejos, Catalina A. and Marioni, Riccardo E.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NPJ DM</abbr></div> <div id="Liley_2024" class="col-sm-8"> <div class="title">Development and assessment of a machine learning tool for predicting emergency admission in Scotland</div> <div class="author">
<a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a>, Gergo Bohner, Samuel R. Emerson, Bilal A. Mateen, Katie Borland, David Carr, Scott Heald, Samuel D. Oduro, Jill Ireland, Keith Moffat, Rachel Porteous, Stephen Riddell, Simon Rogers,  <em>Ioanna Thoma</em>, Nathan Cunningham, Chris Holmes, Katrina Payne, Sebastian J. Vollmer,  <em>Catalina A. Vallejos</em>, and <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a> </div> <div class="periodical"> <em>npj Digital Medicine</em> Oct 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1038/s41746-024-01250-1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Emergency admissions (EA), where a patient requires urgent in-hospital care, are a major challenge for healthcare systems. The development of risk prediction models can partly alleviate this problem by supporting primary care interventions and public health planning. Here, we introduce SPARRAv4, a predictive score for EA risk that will be deployed nationwide in Scotland. SPARRAv4 was derived using supervised and unsupervised machine-learning methods applied to routinely collected electronic health records from approximately 4.8M Scottish residents (2013-18). We demonstrate improvements in discrimination and calibration with respect to previous scores deployed in Scotland, as well as stability over a 3-year timeframe. Our analysis also provides insights about the epidemiology of EA risk in Scotland, by studying predictive performance across different population sub-groups and reasons for admission, as well as by quantifying the effect of individual input features. Finally, we discuss broader challenges including reproducibility and how to safely update risk prediction models that are already deployed at population level.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Liley_2024</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NPJ DM}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development and assessment of a machine learning tool for predicting emergency admission in Scotland}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2398-6352}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1038/s41746-024-01250-1}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41746-024-01250-1}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{npj Digital Medicine}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media LLC}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liley, James and Bohner, Gergo and Emerson, Samuel R. and Mateen, Bilal A. and Borland, Katie and Carr, David and Heald, Scott and Oduro, Samuel D. and Ireland, Jill and Moffat, Keith and Porteous, Rachel and Riddell, Stephen and Rogers, Simon and Thoma, Ioanna and Cunningham, Nathan and Holmes, Chris and Payne, Katrina and Vollmer, Sebastian J. and Vallejos, Catalina A. and Aslett, Louis J. M.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">AI &amp; Ethics</abbr></div> <div id="Chislett_2024" class="col-sm-8"> <div class="title">Ethical considerations of use of hold-out sets in clinical prediction model management</div> <div class="author"> <em>Louis Chislett</em>, <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a>, Alisha R. Davies,  <em>Catalina A. Vallejos</em>, and <a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a> </div> <div class="periodical"> <em>AI and Ethics</em> Sep 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1007/s43681-024-00561-z" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Clinical prediction models are statistical or machine learning models used to quantify the risk of a certain health outcome using patient data. These can then inform potential interventions on patients, causing an effect called performative prediction: predictions inform interventions which influence the outcome they were trying to predict, leading to a potential underestimation of risk in some patients if a model is updated on this data. One suggested resolution to this is the use of hold-out sets, in which a set of patients do not receive model derived risk scores, such that a model can be safely retrained. We present an overview of clinical and research ethics regarding potential implementation of hold-out sets for clinical prediction models in health settings. We focus on the ethical principles of beneficence, non-maleficence, autonomy and justice. We also discuss informed consent, clinical equipoise, and truth-telling. We present illustrative cases of potential hold-out set implementations and discuss statistical issues arising from different hold-out set sampling methods. We also discuss differences between hold-out sets and randomised control trials, in terms of ethics and statistical issues. Finally, we give practical recommendations for researchers interested in the use hold-out sets for clinical prediction models.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Chislett_2024</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{AI &amp; Ethics}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ethical considerations of use of hold-out sets in clinical prediction model management}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2730-5961}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1007/s43681-024-00561-z}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s43681-024-00561-z}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{AI and Ethics}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media LLC}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chislett, Louis and Aslett, Louis J. M. and Davies, Alisha R. and Vallejos, Catalina A. and Liley, James}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PLOS DH</abbr></div> <div id="Thoma_2024" class="col-sm-8"> <div class="title">Differential behaviour of a risk score for emergency hospital admission by demographics in Scotland—A retrospective study</div> <div class="author"> <em>Ioanna Thoma</em>, Simon Rogers, Jillian Ireland, Rachel Porteous, Katie Borland,  <em>Catalina A. Vallejos</em>, <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a>, and <a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a> </div> <div class="periodical"> <em>PLOS Digital Health</em> Dec 2024 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1371/journal.pdig.0000675" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Thoma_2024</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{PLOS DH}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Differential behaviour of a risk score for emergency hospital admission by demographics in Scotland—A retrospective study}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2767-3170}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1371/journal.pdig.0000675}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pdig.0000675}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Digital Health}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science (PLoS)}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Thoma, Ioanna and Rogers, Simon and Ireland, Jillian and Porteous, Rachel and Borland, Katie and Vallejos, Catalina A. and Aslett, Louis J. M. and Liley, James}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Kuo, Po-Chih}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e0000675}</span><span class="p">,</span>
  <span class="na">abtract</span> <span class="p">=</span> <span class="s">{The Scottish Patients at Risk of Re-Admission and Admission (SPARRA) score predicts individual risk of emergency hospital admission for approximately 80% of the Scottish population. It was developed using routinely collected electronic health records, and is used by primary care practitioners to inform anticipatory care, particularly for individuals with high healthcare needs. We comprehensively assess the SPARRA score across population subgroups defined by age, sex, ethnicity, socioeconomic deprivation, and geographic location. For these subgroups, we consider differences in overall performance, score distribution, and false positive and negative rates, using causal methods to identify effects mediated through age, sex, and deprivation. We show that the score is well-calibrated across subgroups, but that rates of false positives and negatives vary widely, mediated by various causes including variability in demographic characteristics, admission reasons, and potentially differential data availability. Our work assists practitioners in the application and interpretation of the SPARRA score in population subgroups.}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Biomet</abbr></div> <div id="https://doi.org/10.1002/bimj.202300060" class="col-sm-8"> <div class="title">A review on statistical and machine learning competing risks methods</div> <div class="author"> <em>Karla Monterrubio-Gómez</em>,  <em>Nathan Constantine-Cooke</em>, and <em>Catalina A. Vallejos</em> </div> <div class="periodical"> <em>Biometrical Journal</em> Feb 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.202300060" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Abstract When modeling competing risks (CR) survival data, several techniques have been proposed in both the statistical and machine learning literature. State-of-the-art methods have extended classical approaches with more flexible assumptions that can improve predictive performance, allow high-dimensional data and missing values, among others. Despite this, modern approaches have not been widely employed in applied settings. This article aims to aid the uptake of such methods by providing a condensed compendium of CR survival methods with a unified notation and interpretation across approaches. We highlight available software and, when possible, demonstrate their usage via reproducible R vignettes. Moreover, we discuss two major concerns that can affect benchmark studies in this context: the choice of performance metrics and reproducibility.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">https://doi.org/10.1002/bimj.202300060</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Biomet}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Monterrubio-Gómez, Karla and Constantine-Cooke, Nathan and Vallejos, Catalina A.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A review on statistical and machine learning competing risks methods}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biometrical Journal}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{66}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2300060}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{competing risks, risk prediction, survival analysis, time-to-event data}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/pdf/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%63%61%74%61%6C%69%6E%61.%76%61%6C%6C%65%6A%6F%73@%69%67%6D%6D.%65%64.%61%63.%75%6B" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0003-3638-1960" title="ORCID" target="_blank" rel="noopener noreferrer"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=lkdrwm0AAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/VallejosGroup" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/catalina-vallejos-61b41956" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a> <a href="https://twitter.com/CataVallejosM" title="Twitter" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a> </div> <div class="contact-note"> </div> </div>  </div> </div> <footer class="fixed-bottom"> <div class="container mt-0" style="text-align: right;"> © Copyright 2025 </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script src="/assets/js/zoom.js"></script> <script src="/assets/js/common.js"></script> <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </article>
</div> </div>
</div></body> </html>
