<!DOCTYPE html> <html lang="en" class=> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> | Publications</title> <meta name="author" content=" "/> <meta name="description" content="The Vallejos Research Group Website "/> <meta name="keywords" content="Bayesian Single-cell RNA-seq EHR"/> <meta property="og:site_name" content=" "/> <meta property="og:type" content="website"/> <meta property="og:title" content=" | Publications"/> <meta property="og:url" content="https://vallejosgroup.github.io/publications/"/> <meta property="og:description" content="The Vallejos Research Group Website "/> <meta property="og:image" content="/assets/img/open-graph-logo.png"/> <meta property="og:locale" content="en"/> <meta name="twitter:card" content="summary"/> <meta name="twitter:title" content="Publications"/> <meta name="twitter:description" content="The Vallejos Research Group Website "/> <meta name="twitter:image" content="/assets/img/open-graph-logo.png"/> <meta name="twitter:site" content="@CataVallejosM"/> <meta name="twitter:creator" content="@CataVallejosM"/> <script type="application/ld+json">
      {
        "author":
        {
          "@type": "Person",
          "name": "  "
        },
        "url": "https://vallejosgroup.github.io/publications/",
        "@type": "WebSite",
        "description": "The Vallejos Research Group Website
",
        "headline": "Publications",
        "sameAs": ["https://orcid.org/0000-0003-3638-1960", "https://scholar.google.com/citations?user=lkdrwm0AAAAJ", "https://github.com/VallejosGroup", "https://www.linkedin.com/in/catalina-vallejos-61b41956", "https://twitter.com/CataVallejosM"],
        "name": "  ",
        "@context": "https://schema.org"
      }
    </script> <link rel="shortcut icon" href="/assets/img/favicon.png"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://vallejosgroup.github.io/publications/"> <link rel="apple-touch-icon" href="https://vallejosgroup.github.io/assets/img/favicon.png"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <div class="font-sans"> <header> <nav id="navbar" class=" navbar font-sans font-light bg-slate-900/30 border-b-2 border-gray-400 border-transparent backdrop-blur-sm navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-light" href="https://vallejosgroup.github.io/"><picture> <source srcset="/assets/img/logo-480.webp" type="image/webp"> <source srcset="/assets/img/logo.png" type="image/png"> <img src="/assets/img/logo" alt="" width="100" height="56"> </source></source></picture></a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link navbarcolor" href="/"> About</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/people/">People</a> </li> <li class="nav-item active"> <a class="nav-link navbarcolor" href="/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/software/">Software</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/contact/">Contact Us</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="text-lg font-sans font-extralight">Publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="year">2025</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">medRxiv</abbr></div> <div id="Constantine-Cooke2024.11.08.24316916" class="col-sm-8"> <div class="title">Large-scale clustering of longitudinal faecal calprotectin and C-reactive protein profiles in inflammatory bowel disease</div> <div class="author"> <em>Nathan Constantine-Cooke</em>, Marie Vibeke Vestergaard, Nikolas Plevris,  <em>Karla Monterrubio-Gómez</em>, Clara Ramos Belinchón, Solomon Ong, Alexander T. Elford, Beatriz Gros, Aleksejs Sazonovs, Gareth-Rhys Jones, Tine Jess,  <em>Catalina A. Vallejos</em>, and <a href="https://charlielees.com" target="_blank" rel="noopener noreferrer">Charlie W. Lees</a> </div> <div class="periodical"> <em>medRxiv</em> Jan 2025 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.medrxiv.org/content/early/2025/01/17/2024.11.08.24316916" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background Crohn’s disease (CD) and ulcerative colitis (UC) are highly heterogeneous, dynamic and unpredictable, with a marked disconnect between symptoms and intestinal inflammation. Attempts to classify inflammatory bowel disease (IBD) subphenotypes to inform clinical decision making have been limited. We aimed to describe the latent disease heterogeneity by modelling routinely collected faecal calprotectin (FC) and CRP data, describing dynamic longitudinal inflammatory patterns in IBD.Methods In this longitudinal study, we analysed patient-level post-diagnosis measurements of FC and CRP in two European cohorts. Latent class mixed models were used to cluster individuals with similar longitudinal profiles. Associations between cluster assignment and baseline characteristics were quantified using multinomial logistic regression. Differences in advanced therapy use across clusters were also explored. Finally, we considered uncertainty in cluster assignments with respect to follow-up length and the overlap between FC and CRP clusters. We included 1036 patients in the FC discovery analysis (Lothian) with a total of 10545 FC observations (median 9 per subject, IQR 6–13), and 7880 patients in the replication (Denmark). The CRP discovery analysis consisted of 1838 patients with 49364 measurements (median 20 per subject; IQR 10–36), with 10041 patients in the replication cohort.Findings Eight distinct clusters of inflammatory behaviour over time were identified in the FC and CRP analysis for the Scottish cohort. This model was then applied to the Danish replication cohort, with similar patterns observed in both the Scottish and Danish populations. The clusters, FC1–8 and CRP1–8, were ordered from the lowest cumulative inflammatory burden to the highest. The clusters included groups with high diagnostic levels of inflammation which rapidly normalised, groups where high inflammation levels persisted throughout the full seven years of observation, and a series of intermediates including delayed remitters and relapsing remitters. CD and UC patients were unevenly distributed across the clusters. In UC, male sex was associated with the poorest prognostic cluster (FC8). The use and timing of advanced therapy was associated with cluster assignment, with the highest use of early advanced therapy in FC1. Of note, FC8 and CRP8 captured consistently high patterns of inflammation despite a high proportion of patients receiving advanced therapy, particularly for CD individuals. We observed that uncertainty in cluster assignments was higher for individuals with short longitudinal follow-up, particularly between clusters capturing similar earlier inflammation patterns. There was broadly poor agreement between FC and CRP clusters in keeping with the need to monitor both in clinical practice.Interpretation Distinct patterns of inflammatory behaviour over time are evident in patients with IBD. Cluster assignment is associated with disease type and both the use and timing of advanced therapy. These data pave the way for a deeper understanding of disease heterogeneity in IBD and enhanced patient stratification in the clinic.Competing Interest StatementNP has served as a speaker for Janssen, Takeda and Pfizer. BG has acted as consultant to Galapagos and Abbvie and as speaker for Abbvie, Jansen, Takeda, Pfizer and Galapagos. GRJ has served as a speaker for Takeda, Janssen, Abbvie, Fresnius and Ferring. TJ has served as a speaker and/or consultant for Ferring and Pfizer. CWL has acted as a speaker and/or consultant to AbbVie, Janssen, Takeda, Pfizer, Galapagos, GSK, Gilead, Vifor Pharma, Ferring, Dr Falk, BMS, Boehringer Ingelheim, Eli Lilly, Merck, Novartis, Sandoz, Celltrion, Cellgene, Amgen, Samsung Bioepis, Fresenius Kabi, Tillotts, Kuma Health, Trellus Health and Iterative Health. None of the other authors report any conflicts of interest.Funding StatementCWL is funded by a UKRI (UK Research and Innovation) Future Leaders Fellowship ’Predicting outcomes in IBD’ (MR/S034919/1). G-RJ is funded by a Wellcome Trust Clinical Research Career Development Fellowship. NC-C was partially supported by the Medical Research Council and The University of Edinburgh via a Precision Medicine PhD studentship (MR/N013166/1). MVV, AS, and TJ are funded by a Center of Excellence Grant (DNRF148) to TJ from the Danish National Research Foundation.Author DeclarationsI confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.YesThe details of the IRB/oversight body that provided approval or exemption for the research described are given below:Usage of the Scottish dataset was approved by the local Caldicott Guardian (Project ID: CRD18002, registered NHS Lothian information asset #IAR-954). In Denmark, studies based on registry data alone are not required to obtain permission from the regional ethics committees as confirmed by The Central Denmark Region Committees on Health Research Ethics (legislation: 1-10-72-148-19)I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.YesI understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).YesI have followed all appropriate research reporting guidelines, such as any relevant EQUATOR Network research reporting checklist(s) and other pertinent material, if applicable.YesAs the data collected for this study has been derived from unconsented patient data, it is not possible to share subject-level data with external entities. Detailed summary level data is available online at https://vallejosgroup.github.io/Lothian-IBDR. The code used to conduct the analysis is also publicly available (https://github.com/VallejosGroup/Lothian-IBDR). https://vallejosgroup.github.io/IBD-Inflammatory-Patterns/ https://github.com/VallejosGroup/IBD-Inflammatory-Patterns</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Constantine-Cooke2024.11.08.24316916</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Constantine-Cooke, Nathan and Vestergaard, Marie Vibeke and Plevris, Nikolas and Monterrubio-G{\'o}mez, Karla and Ramos Belinch{\'o}n, Clara and Ong, Solomon and Elford, Alexander T. and Gros, Beatriz and Sazonovs, Aleksejs and Jones, Gareth-Rhys and Jess, Tine and Vallejos, Catalina A. and Lees, Charlie W.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Large-scale clustering of longitudinal faecal calprotectin and C-reactive protein profiles in inflammatory bowel disease}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2024.11.08.24316916}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2024.11.08.24316916}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory Press}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2025/01/17/2024.11.08.24316916}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2025/01/17/2024.11.08.24316916.full.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Clin Epigen</abbr></div> <div id="Bernabeu_2025" class="col-sm-8"> <div class="title">Blood-based epigenome-wide association study and prediction of alcohol consumption</div> <div class="author"> <em>Elena Bernabeu</em>, Aleksandra D. Chybowska, Jacob K. Kresovich, Matthew Suderman, Daniel L. McCartney, Robert F. Hillary, Janie Corley, Maria Del C. Valdés-Hernández, Susana Muñoz Maniega, Mark E. Bastin, Joanna M. Wardlaw, Zongli Xu, Dale P. Sandler, Archie Campbell, Sarah E. Harris, Andrew M. McIntosh, Jack A. Taylor, Paul Yousefi, Simon R. Cox, Kathryn L. Evans, Matthew R. Robinson,  <em>Catalina A. Vallejos</em>, and Riccardo E. Marioni </div> <div class="periodical"> <em>Clinical Epigenetics</em> Jan 2025 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1186/s13148-025-01818-y" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Alcohol consumption is an important risk factor for multiple diseases. It is typically assessed via self-report, which is open to measurement error through recall bias. Instead, molecular data such as blood-based DNA methylation (DNAm) could be used to derive a more objective measure of alcohol consumption by incorporating information from cytosine-phosphate-guanine (CpG) sites known to be linked to the trait. Here, we explore the epigenetic architecture of self-reported weekly units of alcohol consumption in the Generation Scotland study. We first create a blood-based epigenetic score (EpiScore) of alcohol consumption using elastic net penalized linear regression. We explore the effect of pre-filtering for CpG features ahead of elastic net, as well as differential patterns by sex and by units consumed in the last week relative to an average week. The final EpiScore was trained on 16,717 individuals and tested in four external cohorts: the Lothian Birth Cohorts (LBC) of 1921 and 1936, the Sister Study, and the Avon Longitudinal Study of Parents and Children (total N across studies &gt; 10,000). The maximum Pearson correlation between the EpiScore and self-reported alcohol consumption within cohort ranged from 0.41 to 0.53. In LBC1936, higher EpiScore levels had significant associations with poorer global brain imaging metrics, whereas self-reported alcohol consumption did not. Finally, we identified two novel CpG loci via a Bayesian penalized regression epigenome-wide association study of alcohol consumption. Together, these findings show how DNAm can objectively characterize patterns of alcohol consumption that associate with brain health, unlike self-reported estimates.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Bernabeu_2025</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Clin Epigen}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Blood-based epigenome-wide association study and prediction of alcohol consumption}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1868-7083}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1186/s13148-025-01818-y}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13148-025-01818-y}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Epigenetics}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media LLC}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Bernabeu, Elena and Chybowska, Aleksandra D. and Kresovich, Jacob K. and Suderman, Matthew and McCartney, Daniel L. and Hillary, Robert F. and Corley, Janie and Valdés-Hernández, Maria Del C. and Maniega, Susana Muñoz and Bastin, Mark E. and Wardlaw, Joanna M. and Xu, Zongli and Sandler, Dale P. and Campbell, Archie and Harris, Sarah E. and McIntosh, Andrew M. and Taylor, Jack A. and Yousefi, Paul and Cox, Simon R. and Evans, Kathryn L. and Robinson, Matthew R. and Vallejos, Catalina A. and Marioni, Riccardo E.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2024</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NPJ DM</abbr></div> <div id="Liley_2024" class="col-sm-8"> <div class="title">Development and assessment of a machine learning tool for predicting emergency admission in Scotland</div> <div class="author">
<a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a>, Gergo Bohner, Samuel R. Emerson, Bilal A. Mateen, Katie Borland, David Carr, Scott Heald, Samuel D. Oduro, Jill Ireland, Keith Moffat, Rachel Porteous, Stephen Riddell, Simon Rogers,  <em>Ioanna Thoma</em>, Nathan Cunningham, Chris Holmes, Katrina Payne, Sebastian J. Vollmer,  <em>Catalina A. Vallejos</em>, and <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a> </div> <div class="periodical"> <em>npj Digital Medicine</em> Oct 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1038/s41746-024-01250-1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Emergency admissions (EA), where a patient requires urgent in-hospital care, are a major challenge for healthcare systems. The development of risk prediction models can partly alleviate this problem by supporting primary care interventions and public health planning. Here, we introduce SPARRAv4, a predictive score for EA risk that will be deployed nationwide in Scotland. SPARRAv4 was derived using supervised and unsupervised machine-learning methods applied to routinely collected electronic health records from approximately 4.8M Scottish residents (2013-18). We demonstrate improvements in discrimination and calibration with respect to previous scores deployed in Scotland, as well as stability over a 3-year timeframe. Our analysis also provides insights about the epidemiology of EA risk in Scotland, by studying predictive performance across different population sub-groups and reasons for admission, as well as by quantifying the effect of individual input features. Finally, we discuss broader challenges including reproducibility and how to safely update risk prediction models that are already deployed at population level.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Liley_2024</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NPJ DM}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development and assessment of a machine learning tool for predicting emergency admission in Scotland}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2398-6352}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1038/s41746-024-01250-1}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41746-024-01250-1}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{npj Digital Medicine}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media LLC}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liley, James and Bohner, Gergo and Emerson, Samuel R. and Mateen, Bilal A. and Borland, Katie and Carr, David and Heald, Scott and Oduro, Samuel D. and Ireland, Jill and Moffat, Keith and Porteous, Rachel and Riddell, Stephen and Rogers, Simon and Thoma, Ioanna and Cunningham, Nathan and Holmes, Chris and Payne, Katrina and Vollmer, Sebastian J. and Vallejos, Catalina A. and Aslett, Louis J. M.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">AI &amp; Ethics</abbr></div> <div id="Chislett_2024" class="col-sm-8"> <div class="title">Ethical considerations of use of hold-out sets in clinical prediction model management</div> <div class="author"> <em>Louis Chislett</em>, <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a>, Alisha R. Davies,  <em>Catalina A. Vallejos</em>, and <a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a> </div> <div class="periodical"> <em>AI and Ethics</em> Sep 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1007/s43681-024-00561-z" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Clinical prediction models are statistical or machine learning models used to quantify the risk of a certain health outcome using patient data. These can then inform potential interventions on patients, causing an effect called performative prediction: predictions inform interventions which influence the outcome they were trying to predict, leading to a potential underestimation of risk in some patients if a model is updated on this data. One suggested resolution to this is the use of hold-out sets, in which a set of patients do not receive model derived risk scores, such that a model can be safely retrained. We present an overview of clinical and research ethics regarding potential implementation of hold-out sets for clinical prediction models in health settings. We focus on the ethical principles of beneficence, non-maleficence, autonomy and justice. We also discuss informed consent, clinical equipoise, and truth-telling. We present illustrative cases of potential hold-out set implementations and discuss statistical issues arising from different hold-out set sampling methods. We also discuss differences between hold-out sets and randomised control trials, in terms of ethics and statistical issues. Finally, we give practical recommendations for researchers interested in the use hold-out sets for clinical prediction models.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Chislett_2024</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{AI &amp; Ethics}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ethical considerations of use of hold-out sets in clinical prediction model management}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2730-5961}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1007/s43681-024-00561-z}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s43681-024-00561-z}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{AI and Ethics}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media LLC}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chislett, Louis and Aslett, Louis J. M. and Davies, Alisha R. and Vallejos, Catalina A. and Liley, James}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PLOS DH</abbr></div> <div id="Thoma_2024" class="col-sm-8"> <div class="title">Differential behaviour of a risk score for emergency hospital admission by demographics in Scotland—A retrospective study</div> <div class="author"> <em>Ioanna Thoma</em>, Simon Rogers, Jillian Ireland, Rachel Porteous, Katie Borland,  <em>Catalina A. Vallejos</em>, <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a>, and <a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a> </div> <div class="periodical"> <em>PLOS Digital Health</em> Dec 2024 </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://dx.doi.org/10.1371/journal.pdig.0000675" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Thoma_2024</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{PLOS DH}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Differential behaviour of a risk score for emergency hospital admission by demographics in Scotland—A retrospective study}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2767-3170}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1371/journal.pdig.0000675}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pdig.0000675}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Digital Health}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science (PLoS)}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Thoma, Ioanna and Rogers, Simon and Ireland, Jillian and Porteous, Rachel and Borland, Katie and Vallejos, Catalina A. and Aslett, Louis J. M. and Liley, James}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Kuo, Po-Chih}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e0000675}</span><span class="p">,</span>
  <span class="na">abtract</span> <span class="p">=</span> <span class="s">{The Scottish Patients at Risk of Re-Admission and Admission (SPARRA) score predicts individual risk of emergency hospital admission for approximately 80% of the Scottish population. It was developed using routinely collected electronic health records, and is used by primary care practitioners to inform anticipatory care, particularly for individuals with high healthcare needs. We comprehensively assess the SPARRA score across population subgroups defined by age, sex, ethnicity, socioeconomic deprivation, and geographic location. For these subgroups, we consider differences in overall performance, score distribution, and false positive and negative rates, using causal methods to identify effects mediated through age, sex, and deprivation. We show that the score is well-calibrated across subgroups, but that rates of false positives and negatives vary widely, mediated by various causes including variability in demographic characteristics, admission reasons, and potentially differential data availability. Our work assists practitioners in the application and interpretation of the SPARRA score in population subgroups.}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Biomet</abbr></div> <div id="https://doi.org/10.1002/bimj.202300060" class="col-sm-8"> <div class="title">A review on statistical and machine learning competing risks methods</div> <div class="author"> <em>Karla Monterrubio-Gómez</em>,  <em>Nathan Constantine-Cooke</em>, and <em>Catalina A. Vallejos</em> </div> <div class="periodical"> <em>Biometrical Journal</em> Feb 2024 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.202300060" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Abstract When modeling competing risks (CR) survival data, several techniques have been proposed in both the statistical and machine learning literature. State-of-the-art methods have extended classical approaches with more flexible assumptions that can improve predictive performance, allow high-dimensional data and missing values, among others. Despite this, modern approaches have not been widely employed in applied settings. This article aims to aid the uptake of such methods by providing a condensed compendium of CR survival methods with a unified notation and interpretation across approaches. We highlight available software and, when possible, demonstrate their usage via reproducible R vignettes. Moreover, we discuss two major concerns that can affect benchmark studies in this context: the choice of performance metrics and reproducibility.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">https://doi.org/10.1002/bimj.202300060</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Biomet}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Monterrubio-Gómez, Karla and Constantine-Cooke, Nathan and Vallejos, Catalina A.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A review on statistical and machine learning competing risks methods}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biometrical Journal}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{66}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2300060}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{competing risks, risk prediction, survival analysis, time-to-event data}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/pdf/10.1002/bimj.202300060}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2023</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CGH</abbr></div> <div id="Constantine-Cooke2023" class="col-sm-8"> <div class="title">Longitudinal Fecal Calprotectin Profiles Characterize Disease Course Heterogeneity in Crohn’s Disease</div> <div class="author"> <em>Nathan Constantine-Cooke</em>,  <em>Karla Monterrubio-Gómez</em>, Nikolas Plevris, Lauranne A.A.P. Derikx, Beatriz Gros, Gareth-Rhys Jones, Riccardo E. Marioni, <a href="https://charlielees.com" target="_blank" rel="noopener noreferrer">Charlie W. Lees</a>, and <em>Catalina A. Vallejos</em> </div> <div class="periodical"> <em>Clinical Gastroenterology and Hepatology</em> Oct 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/Constantine-Cooke2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Background and Aims: The progressive nature of Crohn’s disease is highly variable and hard to predict. In addition, symptoms correlate poorly with mucosal inflammation. There is therefore an urgent need to better characterize the heterogeneity of disease trajectories in Crohn’s disease by utilizing objective markers of inflammation. We aimed to better understand this heterogeneity by clustering Crohn’s disease patients with similar longitudinal fecal calprotectin profiles. Methods: We performed a retrospective cohort study at the Edinburgh IBD Unit, a tertiary referral center, and used latent class mixed models to cluster Crohn’s disease subjects using fecal calprotectin observed within 5 years of diagnosis. Information criteria, alluvial plots, and cluster trajectories were used to decide the optimal number of clusters. Chi-square test, Fisher’s exact test, and analysis of variance were used to test for associations with variables commonly assessed at diagnosis. Results: Our study cohort comprised 356 patients with newly diagnosed Crohn’s disease and 2856 fecal calprotectin measurements taken within 5 years of diagnosis (median 7 per subject). Four distinct clusters were identified by characteristic calprotectin profiles: a cluster with consistently high fecal calprotectin and 3 clusters characterized by different downward longitudinal trends. Cluster membership was significantly associated with smoking (P = .015), upper gastrointestinal involvement (P &lt; .001), and early biologic therapy (P &lt; .001). Conclusions: Our analysis demonstrates a novel approach to characterizing the heterogeneity of Crohn’s disease by using fecal calprotectin. The group profiles do not simply reflect different treatment regimens and do not mirror classical disease progression endpoints.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Constantine-Cooke2023</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{CGH}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{Constantine-Cooke2023.pdf}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Longitudinal Fecal Calprotectin Profiles Characterize Disease Course Heterogeneity in {{Crohn}}'s Disease}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{{Constantine-Cooke}, Nathan and {Monterrubio-G{\'o}mez}, Karla and Plevris, Nikolas and Derikx, Lauranne A.A.P. and Gros, Beatriz and Jones, Gareth-Rhys and Marioni, Riccardo E. and Lees, Charlie W. and Vallejos, Catalina A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Clinical Gastroenterology and Hepatology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2918-2927.e6}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{{Elsevier}}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1542-3565}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cgh.2023.03.026}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-05-24}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NatAge</abbr></div> <div id="Cheng_2023" class="col-sm-8"> <div class="title">Development and validation of DNA methylation scores in two European cohorts augment 10-year risk prediction of type 2 diabetes</div> <div class="author"> <em>Yipeng Cheng</em>, Danni A. Gadd, Christian Gieger,  <em>Karla Monterrubio-Gómez</em>, Yufei Zhang, Imrich Berta, Michael J. Stam, Natalia Szlachetka, Evgenii Lobzaev, Nicola Wrobel, Lee Murphy, Archie Campbell, Cliff Nangle, Rosie M. Walker, Chloe Fawns-Ritchie, Annette Peters, Wolfgang Rathmann, David J. Porteous, Kathryn L. Evans, Andrew M. McIntosh, Timothy I. Cannings, Melanie Waldenberger, Andrea Ganna, Daniel L. McCartney,  <em>Catalina A. Vallejos</em>, and Riccardo E. Marioni </div> <div class="periodical"> <em>Nature Aging</em> Apr 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038%2Fs43587-023-00391-4" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> <a href="/assets/pdf/cheng2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Type 2 diabetes mellitus (T2D) presents a major health and economic burden that could be alleviated with improved early prediction and intervention. While standard risk factors have shown good predictive performance, we show that the use of blood-based DNA methylation information leads to a significant improvement in the prediction of 10-year T2D incidence risk. Previous studies have been largely constrained by linear assumptions, the use of cytosine–guanine pairs one-at-a-time and binary outcomes. We present a flexible approach (via an R package, MethylPipeR) based on a range of linear and tree-ensemble models that incorporate time-to-event data for prediction. Using the Generation Scotland cohort (training set ncases = 374, ncontrols = 9,461; test set ncases = 252, ncontrols = 4,526) our best-performing model (area under the receiver operating characteristic curve (AUC) = 0.872, area under the precision-recall curve (PRAUC) = 0.302) showed notable improvement in 10-year onset prediction beyond standard risk factors (AUC = 0.839, precision-recall AUC = 0.227). Replication was observed in the German-based KORA study (n = 1,451, ncases = 142, P = 1.6x10-5).</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Cheng_2023</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s43587-023-00391-4}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038%2Fs43587-023-00391-4}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Science and Business Media {LLC}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{450--458}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Cheng, Yipeng and Gadd, Danni A. and Gieger, Christian and Monterrubio-G{\'{o}}mez, Karla and Zhang, Yufei and Berta, Imrich and Stam, Michael J. and Szlachetka, Natalia and Lobzaev, Evgenii and Wrobel, Nicola and Murphy, Lee and Campbell, Archie and Nangle, Cliff and Walker, Rosie M. and Fawns-Ritchie, Chloe and Peters, Annette and Rathmann, Wolfgang and Porteous, David J. and Evans, Kathryn L. and McIntosh, Andrew M. and Cannings, Timothy I. and Waldenberger, Melanie and Ganna, Andrea and McCartney, Daniel L. and Vallejos, Catalina A. and Marioni, Riccardo E.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development and validation of {DNA} methylation scores in two European cohorts augment 10-year risk prediction of type 2 diabetes}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Aging}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NatAge}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{cheng2021.pdf}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JACC</abbr></div> <div id="doi:10.1016/j.jacc.2022.10.025" class="col-sm-8"> <div class="title">Improving Risk Stratification for Patients With Type 2 Myocardial Infarction</div> <div class="author">Caelan Taggart,  <em>Karla Monterrubio-Gómez</em>, Andreas Roos, Jasper Boeddinghaus, Dorien M. Kimenai, Erik Kadesjo, Anda Bularga, Ryan Wereski, Amy Ferry, Matthew Lowry, Atul Anand, Kuan Ken Lee, Dimitrios Doudesis, Ioanna Manolopoulou, Thomas Nestelberger, Luca Koechlin, Pedro Lopez-Ayala, Christian Mueller, Nicholas L. Mills,  <em>Catalina A. Vallejos</em>, and Andrew R. Chapman </div> <div class="periodical"> <em>Journal of the American College of Cardiology</em> Jan 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.jacc.org/doi/abs/10.1016/j.jacc.2022.10.025" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background: Despite poor cardiovascular outcomes, there are no dedicated, validated risk stratification tools to guide investigation or treatment in type 2 myocardial infarction. Objectives: The goal of this study was to derive and validate a risk stratification tool for the prediction of death or future myocardial infarction in patients with type 2 myocardial infarction. Methods: The T2-risk score was developed in a prospective multicenter cohort of consecutive patients with type 2 myocardial infarction. Cox proportional hazards models were constructed for the primary outcome of myocardial infarction or death at 1 year using variables selected a priori based on clinical importance. Discrimination was assessed by area under the receiving-operating characteristic curve (AUC). Calibration was investigated graphically. The tool was validated in a single-center cohort of consecutive patients and in a multicenter cohort study from sites across Europe. Results: There were 1,121, 250, and 253 patients in the derivation, single-center, and multicenter validation cohorts, with the primary outcome occurring in 27% (297 of 1,121), 26% (66 of 250), and 14% (35 of 253) of patients, respectively. The T2-risk score incorporating age, ischemic heart disease, heart failure, diabetes mellitus, myocardial ischemia on electrocardiogram, heart rate, anemia, estimated glomerular filtration rate, and maximal cardiac troponin concentration had good discrimination (AUC: 0.76; 95% CI: 0.73-0.79) for the primary outcome and was well calibrated. Discrimination was similar in the consecutive patient (AUC: 0.83; 95% CI: 0.77-0.88) and multicenter (AUC: 0.74; 95% CI: 0.64-0.83) cohorts. T2-risk provided improved discrimination over the Global Registry of Acute Coronary Events 2.0 risk score in all cohorts. Conclusions: The T2-risk score performed well in different health care settings and could help clinicians to prognosticate, as well as target investigation and preventative therapies more effectively. (High-Sensitivity Troponin in the Evaluation of Patients With Suspected Acute Coronary Syndrome [High-STEACS]; NCT01852123)</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">doi:10.1016/j.jacc.2022.10.025</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Taggart, Caelan and Monterrubio-Gómez, Karla and Roos, Andreas and Boeddinghaus, Jasper and Kimenai, Dorien M. and Kadesjo, Erik and Bularga, Anda and Wereski, Ryan and Ferry, Amy and Lowry, Matthew and Anand, Atul and Lee, Kuan Ken and Doudesis, Dimitrios and Manolopoulou, Ioanna and Nestelberger, Thomas and Koechlin, Luca and Lopez-Ayala, Pedro and Mueller, Christian and Mills, Nicholas L. and Vallejos, Catalina A. and Chapman, Andrew R.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Improving Risk Stratification for Patients With Type 2 Myocardial Infarction}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of the American College of Cardiology}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{JACC}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{81}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{156-168}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.jacc.2022.10.025}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.jacc.org/doi/abs/10.1016/j.jacc.2022.10.025}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.jacc.org/doi/pdf/10.1016/j.jacc.2022.10.025}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2021</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Kapourani2021" class="col-sm-8"> <div class="title">scMET: Bayesian modeling of DNA methylation heterogeneity at single-cell resolution</div> <div class="author"> <em>Chantriolnt-Andreas Kapourani</em>, Ricard Argelaguet, <a href="https://datascience.sissa.it/person/11/guido-sanguinetti" target="_blank" rel="noopener noreferrer">Guido Sanguinetti</a>, and <em>Catalina A Vallejos</em> </div> <div class="periodical"> <em>Genome Biology</em> Jan 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-021-02329-8" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>High-throughput single-cell measurements of DNA methylomes can quantify methylation heterogeneity and uncover its role in gene regulation. However, technical limitations and sparse coverage can preclude this task. scMET is a hierarchical Bayesian model which overcomes sparsity, sharing information across cells and genomic features to robustly quantify genuine biological heterogeneity. scMET can identify highly variable features that drive epigenetic heterogeneity, and perform differential methylation and variability analyses. We illustrate how scMET facilitates the characterization of epigenetically distinct cell populations and how it enables the formulation of novel hypotheses on the epigenetic regulation of gene expression. scMET is available at https://github.com/andreaskapou/scMET.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Kapourani2021</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kapourani, Chantriolnt-Andreas and Argelaguet, Ricard and Sanguinetti, Guido and Vallejos, Catalina A}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2021/04/20}</span><span class="p">,</span>
  <span class="na">date-added</span> <span class="p">=</span> <span class="s">{2022-02-26 12:43:46 +0000}</span><span class="p">,</span>
  <span class="na">date-modified</span> <span class="p">=</span> <span class="s">{2022-02-26 12:43:46 +0000}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Kapourani2021}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{114}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{scMET: Bayesian modeling of DNA methylation heterogeneity at single-cell resolution}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="liley2021" class="col-sm-8"> <div class="title">Model updating after interventions paradoxically introduces bias</div> <div class="author">
<a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a>, Samuel R Emerson, Bilal A Mateen,  <em>Catalina A Vallejos</em>, Louis J M Aslett, and Sebastian J Vollmer </div> <div class="periodical"> <em>arXiv</em> Jan 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2010.11530" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/liley2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Machine learning is increasingly being used to generate prediction models for use in a number of real-world settings, from credit risk assessment to clinical decision support. Recent discussions have highlighted potential problems in the updating of a predictive score for a binary outcome when an existing predictive score forms part of the standard workflow, driving interventions. In this setting, the existing score induces an additional causative pathway which leads to miscalibration when the original score is replaced. We propose a general causal framework to describe and address this problem, and demonstrate an equivalent formulation as a partially observed Markov decision process. We use this model to demonstrate the impact of such ‘naive updating’ when performed repeatedly. Namely, we show that successive predictive scores may converge to a point where they predict their own effect, or may eventually tend toward a stable oscillation between two values, and we argue that neither outcome is desirable. Furthermore, we demonstrate that even if model-fitting procedures improve, actual performance may worsen. We complement these findings with a discussion of several potential routes to overcome these issues.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liley2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Model updating after interventions paradoxically introduces bias}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{liley2021.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liley, James and Emerson, Samuel R and Mateen, Bilal A and Vallejos, Catalina A and Aslett, Louis J M and Vollmer, Sebastian J}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">arxiv</span> <span class="p">=</span> <span class="s">{2010.11530}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">primaryclass</span> <span class="p">=</span> <span class="s">{stat.ML}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NatCom</abbr></div> <div id="Richter2021" class="col-sm-8"> <div class="title">Single-nucleus RNA-seq2 reveals functional crosstalk between liver zonation and ploidy</div> <div class="author">M. L. Richter, I. K. Deligiannis, K. Yin, A. Danese, E. Lleshi, P. Coupland,  <em>C. A. Vallejos</em>, K. P. Matchett, N. C. Henderson, M. Colome-Tatche, and C. P. Martinez-Jimenez </div> <div class="periodical"> <em>Nature Communications</em> Jan 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038/s41467-021-24543-5" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell RNA-seq reveals the role of pathogenic cell populations in development and progression of chronic diseases. In order to expand our knowledge on cellular heterogeneity, we have developed a single-nucleus RNA-seq2 method tailored for the comprehensive analysis of the nuclear transcriptome from frozen tissues, allowing the dissection of all cell types present in the liver, regardless of cell size or cellular fragility. We use this approach to characterize the transcriptional profile of individual hepatocytes with different levels of ploidy, and have discovered that ploidy states are associated with different metabolic potential, and gene expression in tetraploid mononucleated hepatocytes is conditioned by their position within the hepatic lobule. Our work reveals a remarkable crosstalk between gene dosage and spatial distribution of hepatocytes.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Richter2021</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NatCom}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Richter, M. L. and Deligiannis, I. K. and Yin, K. and Danese, A. and Lleshi, E. and Coupland, P. and Vallejos, C. A. and Matchett, K. P. and Henderson, N. C. and Colome-Tatche, M. and Martinez-Jimenez, C. P.}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2021/07/12}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-021-24543-5}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Richter2021}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{2041-1723}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{4264}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Single-nucleus RNA-seq2 reveals functional crosstalk between liver zonation and ploidy}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/s41467-021-24543-5}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">medRxiv</abbr></div> <div id="Liley2021.08.06.21261593" class="col-sm-8"> <div class="title">Development and assessment of a machine learning tool for predicting emergency admission in Scotland</div> <div class="author">
<a href="https://www.durham.ac.uk/staff/james-liley/" target="_blank" rel="noopener noreferrer">James Liley</a>, Gergo Bohner, Samuel R. Emerson, Bilal A. Mateen, Katie Borland, David Carr, Scott Heald, Samuel D. Oduro, Jill Ireland, Keith Moffat, Rachel Porteous, Stephen Riddell, Nathan Cunningham, Chris Holmes, Katrina Payne, Sebastian J. Vollmer,  <em>Catalina A. Vallejos</em>, and <a href="https://www.durham.ac.uk/staff/louis-aslett/" target="_blank" rel="noopener noreferrer">Louis J. M. Aslett</a> </div> <div class="periodical"> <em>medRxiv</em> Jan 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.medrxiv.org/content/early/2021/08/10/2021.08.06.21261593" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Avoiding emergency hospital admission (EA) is advantageous to individual health and the healthcare system. We develop a statistical model estimating risk of EA for most of the Scottish population (&gt; 4.8M individuals) using electronic health records, such as hospital episodes and prescribing activity. We demonstrate good predictive accuracy (AUROC 0.80), calibration and temporal stability. We find strong prediction of respiratory and metabolic EA, show a substantial risk contribution from socioeconomic decile, and highlight an important problem in model updating. Our work constitutes a rare example of a population-scale machine learning score to be deployed in a healthcare setting.Competing Interest StatementThe authors have declared no competing interest.Funding StatementJL, CAV and LJMA were partially supported by Wave 1 of The UKRI Strategic Priorities Fund under the EPSRC Grant EP/T001569/1, particularly the "Health" theme within that grant and The Alan Turing Institute; JL, BAM, CAV, LJMA and SJV were partially supported by Health Data Research UK, an initiative funded by UK Research and Innovation, Department of Health and Social Care (England), the devolved administrations, and leading medical research charities; SJV, NC and GB were partially supported by the University of Warwick Impact Fund. SRE is funded by the EPSRC doctoral training partnership (DTP) at Durham University, grant reference EP/R513039/1; LJMA was partially supported by a Health Programme Fellowship at The Alan Turing Institute; CAV was supported by a Chancellor’s Fellowship provided by the University of Edinburgh.Author DeclarationsI confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.YesThe details of the IRB/oversight body that provided approval or exemption for the research described are given below:This study and the use of NHS data was approved by the Public Benefit and Privacy Panel for Health and Social Care (study number 1718-0370; approval evidenced in application outcome minutes for 2018/19 at https://www.informationgovernance.scot.nhs.uk/pbpphsc/application-outcomes/ ). In addition, accessing data was approved by the Public Health Scotland National Safe Haven, through the the electronic Data Research and Innovation Service (eDRIS) and the Public Benefit and Privacy Panel (PBPP) (study number 1718-0370). All studies have been conducted in accordance with information governance standards; data had no patient identifiers available to the researchers. This work was conducted in accordance with UK data governance regulations under PBPP application number eDRIS 1718-0370 All necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived.YesI understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).YesI have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.YesRaw data for this project are patient-level NHS Scotland health records, and are confidential. Due to the confidential nature of the data used, all analysis took place on remote ’safe havens’, without access to internet, software updates or unpublished software. Information Governance training was required for all researchers accessing the analysis environment. Moreover, to avoid the risk of accidental disclosure of sensitive information, an independent team carried out statistical disclosure control checks to all data exports, including the outputs presented in this manuscript. All analysis code and co-ordinates required to reproduce our Figures are available in github.com/jamesliley/SPARRAv4 https://github.com/jamesliley/SPARRAv4</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Liley2021.08.06.21261593</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liley, James and Bohner, Gergo and Emerson, Samuel R. and Mateen, Bilal A. and Borland, Katie and Carr, David and Heald, Scott and Oduro, Samuel D. and Ireland, Jill and Moffat, Keith and Porteous, Rachel and Riddell, Stephen and Cunningham, Nathan and Holmes, Chris and Payne, Katrina and Vollmer, Sebastian J. and Vallejos, Catalina A. and Aslett, Louis J. M.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development and assessment of a machine learning tool for predicting emergency admission in Scotland}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2021.08.06.21261593}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2021.08.06.21261593}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory Press}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2021/08/10/2021.08.06.21261593}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2021/08/10/2021.08.06.21261593.full.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div> <div id="Maniatis2021.05.13.443959" class="col-sm-8"> <div class="title">SCRaPL: hierarchical Bayesian modelling of associations in single cell multi-omics data</div> <div class="author"> <em>Christos Maniatis</em>,  <em>Catalina A Vallejos</em>, and <a href="https://datascience.sissa.it/person/11/guido-sanguinetti" target="_blank" rel="noopener noreferrer">Guido Sanguinetti</a> </div> <div class="periodical"> <em>bioRxiv</em> Jan 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/early/2021/05/14/2021.05.13.443959" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell multi-omics assays offer unprecedented opportunities to explore gene regulation at cellular level. However, high levels of technical noise and data sparsity frequently lead to a lack of statistical power in correlative analyses, identifying very few, if any, significant associations between different molecular layers. Here we propose SCRaPL, a novel computational tool that increases power by carefully modelling noise in the experimental systems. We show on real and simulated multi-omics single-cell data sets that SCRaPL achieves higher sensitivity and better robustness in identifying correlations, while maintaining a similar level of false positives as standard analyses based on Pearson correlation.Competing Interest StatementThe authors have declared no competing interest.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Maniatis2021.05.13.443959</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Maniatis, Christos and Vallejos, Catalina A and Sanguinetti, Guido}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SCRaPL: hierarchical Bayesian modelling of associations in single cell multi-omics data}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2021.05.13.443959}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2021.05.13.443959}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/early/2021/05/14/2021.05.13.443959}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/early/2021/05/14/2021.05.13.443959.full.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2020</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Lahnemann2020" class="col-sm-8"> <div class="title">Eleven grand challenges in single-cell data science</div> <div class="author">David Lähnemann, Johannes Köster, Ewa Szczurek, Davis J. McCarthy, Stephanie C. Hicks, Mark D. Robinson,  <em>Catalina A. Vallejos</em>, Kieran R. Campbell, Niko Beerenwinkel, Ahmed Mahfouz, Luca Pinello, Pavel Skums, Alexandros Stamatakis, Camille Stephan-Otto Attolini, Samuel Aparicio, Jasmijn Baaijens, Marleen Balvert, Buys de Barbanson, Antonio Cappuccio, Giacomo Corleone, Bas E. Dutilh, Maria Florescu, Victor Guryev, Rens Holmer, Katharina Jahn, Thamar Jessurun Lobo, Emma M. Keizer, Indu Khatri, Szymon M. Kielbasa, Jan O. Korbel, Alexey M. Kozlov, Tzu-Hao Kuo, Boudewijn P. F. Lelieveldt, Ion I. Mandoiu, <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C. Marioni</a>, Tobias Marschall, Felix Mölder, Amir Niknejad, Lukasz Raczkowski, Marcel Reinders, Jeroen de Ridder, Antoine-Emmanuel Saliba, Antonios Somarakis, Oliver Stegle, Fabian J. Theis, Huan Yang, Alex Zelikovsky, Alice C. McHardy, Benjamin J. Raphael, Sohrab P. Shah, and Alexander Schönhuth </div> <div class="periodical"> <em>Genome Biology</em> Jan 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-020-1926-6" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>The recent boom in microfluidics and combinatorial indexing strategies, combined with low sequencing costs, has empowered single-cell sequencing technology. Thousands—or even millions—of cells analyzed in a single experiment amount to a data revolution in single-cell biology and pose unique data science problems. Here, we outline eleven challenges that will be central to bringing this emerging field of single-cell data science forward. For each challenge, we highlight motivating research questions, review prior work, and formulate open problems. This compendium is for established researchers, newcomers, and students alike, highlighting interesting and rewarding problems for the coming years.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Lahnemann2020</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{L{\"a}hnemann, David and K{\"o}ster, Johannes and Szczurek, Ewa and McCarthy, Davis J. and Hicks, Stephanie C. and Robinson, Mark D. and Vallejos, Catalina A. and Campbell, Kieran R. and Beerenwinkel, Niko and Mahfouz, Ahmed and Pinello, Luca and Skums, Pavel and Stamatakis, Alexandros and Attolini, Camille Stephan-Otto and Aparicio, Samuel and Baaijens, Jasmijn and Balvert, Marleen and Barbanson, Buys de and Cappuccio, Antonio and Corleone, Giacomo and Dutilh, Bas E. and Florescu, Maria and Guryev, Victor and Holmer, Rens and Jahn, Katharina and Lobo, Thamar Jessurun and Keizer, Emma M. and Khatri, Indu and Kielbasa, Szymon M. and Korbel, Jan O. and Kozlov, Alexey M. and Kuo, Tzu-Hao and Lelieveldt, Boudewijn P. F. and Mandoiu, Ion I. and Marioni, John C. and Marschall, Tobias and M{\"o}lder, Felix and Niknejad, Amir and Raczkowski, Lukasz and Reinders, Marcel and Ridder, Jeroen de and Saliba, Antoine-Emmanuel and Somarakis, Antonios and Stegle, Oliver and Theis, Fabian J. and Yang, Huan and Zelikovsky, Alex and McHardy, Alice C. and Raphael, Benjamin J. and Shah, Sohrab P. and Sch{\"o}nhuth, Alexander}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2020/02/07}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-020-1926-6}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{L{\"a}hnemann2020}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{31}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Eleven grand challenges in single-cell data science}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-020-1926-6}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-020-1926-6}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Circ</abbr></div> <div id="doi:10.1161/CIRCULATIONAHA.119.042960" class="col-sm-8"> <div class="title">High-Sensitivity Cardiac Troponin and the Universal Definition of Myocardial Infarction</div> <div class="author">Andrew R. Chapman, Philip D. Adamson, Anoop S.V. Shah, Atul Anand, Fiona E. Strachan, Amy V. Ferry, Kuan Ken Lee, Colin Berry, Iain Findlay, Anne Cruikshank, Alan Reid, Alasdair Gray, Paul O. Collinson, Fred Apple, David A. McAllister, Donogh Maguire, Keith A.A. Fox,  <em>Catalina A. Vallejos</em>, Catriona Keerie, Christopher J. Weir, David E. Newby, Nicholas L. Mills, Christopher Tuck, Anda Bularga, Ryan Wereski, Dennis Sandeman, Catherine L. Stables, Athanasios Tsanasis, Lucy Marshall, Stacey D. Stewart, Takeshi Fujisawa, Mischa Hautvast, Jean McPherson, Lynn McKinlay, Simon Walker, Ian Ford, Simon Walker, Shannon Amoils, Jennifer Stevens, John Norrie, Jack Andrews, Phil Adamson, Alastair Moss, Mohamed Anwar, John Hung, Simon Walker, Jonathan Malo, Colin Fischbacher, Bernard Croal, Stephen J. Leslie, Richard Parker, Allan Walker, Ronnie Harkess, Chris Tuck, Tony Wackett, Roma Armstrong, Marion Flood, Laura Stirling, Claire MacDonald, Imran Sadat, Frank Finlay, Heather Charles, Pamela Linksted, Stephen Young, Bill Alexander, and Chris Duncan </div> <div class="periodical"> <em>Circulation</em> Jan 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.ahajournals.org/doi/abs/10.1161/CIRCULATIONAHA.119.042960" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background: The introduction of more sensitive cardiac troponin assays has led to increased recognition of myocardial injury in acute illnesses other than acute coronary syndrome. The Universal Definition of Myocardial Infarction recommends high-sensitivity cardiac troponin testing and classification of patients with myocardial injury based on pathogenesis, but the clinical implications of implementing this guideline are not well understood. Methods: In a stepped-wedge cluster randomized, controlled trial, we implemented a high-sensitivity cardiac troponin assay and the recommendations of the Universal Definition in 48 282 consecutive patients with suspected acute coronary syndrome. In a prespecified secondary analysis, we compared the primary outcome of myocardial infarction or cardiovascular death and secondary outcome of noncardiovascular death at 1 year across diagnostic categories. Results: Implementation increased the diagnosis of type 1 myocardial infarction by 11% (510/4471), type 2 myocardial infarction by 22% (205/916), and acute and chronic myocardial injury by 36% (443/1233) and 43% (389/898), respectively. Compared with those without myocardial injury, the rate of the primary outcome was highest in those with type 1 myocardial infarction (cause-specific hazard ratio [HR] 5.64 [95% CI, 5.12–6.22]), but was similar across diagnostic categories, whereas noncardiovascular deaths were highest in those with acute myocardial injury (cause specific HR 2.65 [95% CI, 2.33–3.01]). Despite modest increases in antiplatelet therapy and coronary revascularization after implementation in patients with type 1 myocardial infarction, the primary outcome was unchanged (cause specific HR 1.00 [95% CI, 0.82–1.21]). Increased recognition of type 2 myocardial infarction and myocardial injury did not lead to changes in investigation, treatment or outcomes. Conclusions: Implementation of high-sensitivity cardiac troponin assays and the recommendations of the Universal Definition of Myocardial Infarction identified patients at high-risk of cardiovascular and noncardiovascular events but was not associated with consistent increases in treatment or improved outcomes. Trials of secondary prevention are urgently required to determine whether this risk is modifiable in patients without type 1 myocardial infarction.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">doi:10.1161/CIRCULATIONAHA.119.042960</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Circ}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chapman, Andrew R. and Adamson, Philip D. and Shah, Anoop S.V. and Anand, Atul and Strachan, Fiona E. and Ferry, Amy V. and Lee, Kuan Ken and Berry, Colin and Findlay, Iain and Cruikshank, Anne and Reid, Alan and Gray, Alasdair and Collinson, Paul O. and Apple, Fred and McAllister, David A. and Maguire, Donogh and Fox, Keith A.A. and Vallejos, Catalina A. and Keerie, Catriona and Weir, Christopher J. and Newby, David E. and Mills, Nicholas L. and Tuck, Christopher and Bularga, Anda and Wereski, Ryan and Sandeman, Dennis and Stables, Catherine L. and Tsanasis, Athanasios and Marshall, Lucy and Stewart, Stacey D. and Fujisawa, Takeshi and Hautvast, Mischa and McPherson, Jean and McKinlay, Lynn and Walker, Simon and Ford, Ian and Walker, Simon and Amoils, Shannon and Stevens, Jennifer and Norrie, John and Andrews, Jack and Adamson, Phil and Moss, Alastair and Anwar, Mohamed and Hung, John and Walker, Simon and Malo, Jonathan and Fischbacher, Colin and Croal, Bernard and Leslie, Stephen J. and Parker, Richard and Walker, Allan and Harkess, Ronnie and Tuck, Chris and Wackett, Tony and Armstrong, Roma and Flood, Marion and Stirling, Laura and MacDonald, Claire and Sadat, Imran and Finlay, Frank and Charles, Heather and Linksted, Pamela and Young, Stephen and Alexander, Bill and Duncan, Chris}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High-Sensitivity Cardiac Troponin and the Universal Definition of Myocardial Infarction}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Circulation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{141}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{161-171}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1161/CIRCULATIONAHA.119.042960}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.ahajournals.org/doi/abs/10.1161/CIRCULATIONAHA.119.042960}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.ahajournals.org/doi/pdf/10.1161/CIRCULATIONAHA.119.042960}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2018</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CellSys</abbr></div> <div id="Eling2018" class="col-sm-8"> <div class="title">Correcting the Mean-Variance Dependency for Differential Variability Testing Using Single-Cell RNA Sequencing Data</div> <div class="author">Nils Eling, Arianne C Richard, Sylvia Richardson, <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C Marioni</a>, and <em>Catalina A Vallejos</em> </div> <div class="periodical"> <em>Cell Systems</em> Jan 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2405471218302783" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Cell-to-cell transcriptional variability in otherwise homogeneous cell populations plays an important role in tissue function and development. Single-cell RNA sequencing can characterize this variability in a transcriptome-wide manner. However, technical variation and the confounding between variability and mean expression estimates hinder meaningful comparison of expression variability between cell populations. To address this problem, we introduce an analysis approach that extends the BASiCS statistical framework to derive a residual measure of variability that is not confounded by mean expression. This includes a robust procedure for quantifying technical noise in experiments where technical spike-in molecules are not available. We illustrate how our method provides biological insight into the dynamics of cell-to-cell expression variability, highlighting a synchronization of biosynthetic machinery components in immune cells upon activation. In contrast to the uniform up-regulation of the biosynthetic machinery, CD4+ T cells show heterogeneous up-regulation of immune-related and lineage-defining genes during activation and differentiation.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Eling2018</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{CellSys}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Correcting the Mean-Variance Dependency for Differential Variability Testing Using Single-Cell RNA Sequencing Data}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Systems}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{284-294.e12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2405-4712}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.cels.2018.06.011}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S2405471218302783}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Eling, Nils and Richard, Arianne C and Richardson, Sylvia and Marioni, John C and Vallejos, Catalina A}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{single-cell RNA sequencing, transcriptional noise, variability, immune activation, statistics, Bayesian}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Lancet</abbr></div> <div id="SHAH2018919" class="col-sm-8"> <div class="title">High-sensitivity troponin in the evaluation of patients with suspected acute coronary syndrome: a stepped-wedge, cluster-randomised controlled trial</div> <div class="author">Anoop S V Shah, Atul Anand, Fiona E Strachan, Amy V Ferry, Kuan Ken Lee, Andrew R Chapman, Dennis Sandeman, Catherine L Stables, Philip D Adamson, Jack P M Andrews, Mohamed S Anwar, John Hung, Alistair J Moss, Rachel O’Brien, Colin Berry, Iain Findlay, Simon Walker, Anne Cruickshank, Alan Reid, Alasdair Gray, Paul O Collinson, Fred S Apple, David A McAllister, Donogh Maguire, Keith A A Fox, David E Newby, Christopher Tuck, Ronald Harkess, Richard A Parker, Catriona Keerie, Christopher J Weir, Nicholas L Mills, Lucy Marshall, Stacey D Stewart, Takeshi Fujisawa,  <em>Catalina A Vallejos</em>, Athanasios Tsanas, Mischa Hautvast, Jean McPherson, Lynn McKinlay, Jonathan Malo, Colin M Fischbacher, Bernard L Croal, Stephen J Leslie, Allan Walker, Tony Wackett, Roma Armstrong, Laura Stirling, Claire MacDonald, Imran Sadat, Frank Finlay, Heather Charles, Pamela Linksted, Stephen Young, Bill Alexander, and Chris Duncan </div> <div class="periodical"> <em>The Lancet</em> Jan 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0140673618319238" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background: High-sensitivity cardiac troponin assays permit use of lower thresholds for the diagnosis of myocardial infarction, but whether this improves clinical outcomes is unknown. We aimed to determine whether the introduction of a high-sensitivity cardiac troponin I (hs-cTnI) assay with a sex-specific 99th centile diagnostic threshold would reduce subsequent myocardial infarction or cardiovascular death in patients with suspected acute coronary syndrome. Methods: In this stepped-wedge, cluster-randomised controlled trial across ten secondary or tertiary care hospitals in Scotland, we evaluated the implementation of an hs-cTnI assay in consecutive patients who had been admitted to the hospitals’ emergency departments with suspected acute coronary syndrome. Patients were eligible for inclusion if they presented with suspected acute coronary syndrome and had paired cardiac troponin measurements from the standard care and trial assays. During a validation phase of 6–12 months, results from the hs-cTnI assay were concealed from the attending clinician, and a contemporary cardiac troponin I (cTnI) assay was used to guide care. Hospitals were randomly allocated to early (n=5 hospitals) or late (n=5 hospitals) implementation, in which the high-sensitivity assay and sex-specific 99th centile diagnostic threshold was introduced immediately after the 6-month validation phase or was deferred for a further 6 months. Patients reclassified by the high-sensitivity assay were defined as those with an increased hs-cTnI concentration in whom cTnI concentrations were below the diagnostic threshold on the contemporary assay. The primary outcome was subsequent myocardial infarction or death from cardiovascular causes at 1 year after initial presentation. Outcomes were compared in patients reclassified by the high-sensitivity assay before and after its implementation by use of an adjusted generalised linear mixed model. This trial is registered with ClinicalTrials.gov, number NCT01852123. Findings: Between June 10, 2013, and March 3, 2016, we enrolled 48 282 consecutive patients (61 [SD 17] years, 47% women) of whom 10 360 (21%) patients had cTnI concentrations greater than those of the 99th centile of the normal range of values, who were identified by the contemporary assay or the high-sensitivity assay. The high-sensitivity assay reclassified 1771 (17%) of 10 360 patients with myocardial injury or infarction who were not identified by the contemporary assay. In those reclassified, subsequent myocardial infarction or cardiovascular death within 1 year occurred in 105 (15%) of 720 patients in the validation phase and 131 (12%) of 1051 patients in the implementation phase (adjusted odds ratio for implementation vs validation phase 1·10, 95% CI 0·75 to 1·61; p=0·620). Interpretation: Use of a high-sensitivity assay prompted reclassification of 1771 (17%) of 10 360 patients with myocardial injury or infarction, but was not associated with a lower subsequent incidence of myocardial infarction or cardiovascular death at 1 year. Our findings question whether the diagnostic threshold for myocardial infarction should be based on the 99th centile derived from a normal reference population.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">SHAH2018919</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Lancet}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High-sensitivity troponin in the evaluation of patients with suspected acute coronary syndrome: a stepped-wedge, cluster-randomised controlled trial}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The Lancet}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{392}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10151}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{919-928}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0140-6736}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/S0140-6736(18)31923-8}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0140673618319238}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shah, Anoop S V and Anand, Atul and Strachan, Fiona E and Ferry, Amy V and Lee, Kuan Ken and Chapman, Andrew R and Sandeman, Dennis and Stables, Catherine L and Adamson, Philip D and Andrews, Jack P M and Anwar, Mohamed S and Hung, John and Moss, Alistair J and O'Brien, Rachel and Berry, Colin and Findlay, Iain and Walker, Simon and Cruickshank, Anne and Reid, Alan and Gray, Alasdair and Collinson, Paul O and Apple, Fred S and McAllister, David A and Maguire, Donogh and Fox, Keith A A and Newby, David E and Tuck, Christopher and Harkess, Ronald and Parker, Richard A and Keerie, Catriona and Weir, Christopher J and Mills, Nicholas L and Marshall, Lucy and Stewart, Stacey D and Fujisawa, Takeshi and Vallejos, Catalina A and Tsanas, Athanasios and Hautvast, Mischa and McPherson, Jean and McKinlay, Lynn and Malo, Jonathan and Fischbacher, Colin M and Croal, Bernard L and Leslie, Stephen J and Walker, Allan and Wackett, Tony and Armstrong, Roma and Stirling, Laura and MacDonald, Claire and Sadat, Imran and Finlay, Frank and Charles, Heather and Linksted, Pamela and Young, Stephen and Alexander, Bill and Duncan, Chris}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2017</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NatMet</abbr></div> <div id="Vallejos2017" class="col-sm-8"> <div class="title">Normalizing single-cell RNA sequencing data: challenges and opportunities</div> <div class="author"> <em>Catalina A Vallejos</em>, Davide Risso, Antonio Scialdone, Sandrine Dudoit, and <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C Marioni</a> </div> <div class="periodical"> <em>Nature Methods</em> Jan 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038/nmeth.4292" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell transcriptomics is becoming an important component of the molecular biologist’s toolkit. A critical step when analyzing data generated using this technology is normalization. However, normalization is typically performed using methods developed for bulk RNA sequencing or even microarray data, and the suitability of these methods for single-cell transcriptomics has not been assessed. We here discuss commonly used normalization approaches and illustrate how these can produce misleading results. Finally, we present alternative approaches and provide recommendations for single-cell RNA sequencing users.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2017</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NatMet}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Normalizing single-cell RNA sequencing data: challenges and opportunities}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A and Risso, Davide and Scialdone, Antonio and Dudoit, Sandrine and Marioni, John C}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2017/06/01}</span><span class="p">,</span>
  <span class="na">date-added</span> <span class="p">=</span> <span class="s">{2022-02-26 11:24:03 +0000}</span><span class="p">,</span>
  <span class="na">date-modified</span> <span class="p">=</span> <span class="s">{2022-02-26 11:24:03 +0000}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/nmeth.4292}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1548-7105}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Methods}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{565--571}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/nmeth.4292}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">EconStat</abbr></div> <div id="VALLEJOS201773" class="col-sm-8"> <div class="title">Incorporating unobserved heterogeneity in Weibull survival models: A Bayesian approach</div> <div class="author"> <em>Catalina A. Vallejos</em>, and <a href="https://warwick.ac.uk/fac/sci/statistics/staff/academic-research/steel/steel_homepage" target="_blank" rel="noopener noreferrer">Mark F.J. Steel</a> </div> <div class="periodical"> <em>Econometrics and Statistics</em> Jan 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2452306217300072" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Outlying observations and other forms of unobserved heterogeneity can distort inference for survival datasets. The family of Rate Mixtures of Weibull distributions includes subject-level frailty terms as a solution to this issue. With a parametric mixing distribution assigned to the frailties, this family generates flexible hazard functions. Covariates are introduced via an Accelerated Failure Time specification for which the interpretation of the regression coefficients does not depend on the choice of mixing distribution. A weakly informative prior is proposed by combining the structure of the Jeffreys prior with a proper prior on some model parameters. This improper prior is shown to lead to a proper posterior distribution under easily satisfied conditions. By eliciting the proper component of the prior through the coefficient of variation of the survival times, prior information is matched for different mixing distributions. Posterior inference on subject-level frailty terms is exploited as a tool for outlier detection. Finally, the proposed methodology is illustrated using two real datasets, one concerning bone marrow transplants and another on cerebral palsy.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">VALLEJOS201773</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{EconStat}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Incorporating unobserved heterogeneity in Weibull survival models: A Bayesian approach}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Econometrics and Statistics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{73-88}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2452-3062}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.ecosta.2017.01.005}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S2452306217300072}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A. and Steel, Mark F.J.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Survival analysis, Frailty model, Robust modelling, Outlier detection, Posterior existence}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Science</abbr></div> <div id="doi10.1126/science.aah4115" class="col-sm-8"> <div class="title">Aging increases cell-to-cell transcriptional variability upon immune stimulation</div> <div class="author">Celia Pilar Martinez-Jimenez, Nils Eling, Hung-Chang Chen,  <em>Catalina A. Vallejos</em>, Aleksandra A. Kolodziejczyk, Frances Connor, Lovorka Stojic, Timothy F. Rayner, Michael J. T. Stubbington, Sarah A. Teichmann, Maike Roche, <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C. Marioni</a>, and Duncan T. Odom </div> <div class="periodical"> <em>Science</em> Jan 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/abs/10.1126/science.aah4115" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell sequencing of mouse immune cells reveals how aging destabilizes a conserved transcriptional activation program. How and why the immune system becomes less effective with age are not well understood. Martinez-Jimenez et al. performed single-cell sequencing of CD4+ T cells in old and young mice of two species. In young mice, the gene expression program of early immune activation was tightly regulated and conserved between species. However, as mice aged, the expression of genes involved in pathways responding to immune cell stimulation was not as robust and exhibited increased cell-to-cell variability. Science, this issue p. 1433 Aging is characterized by progressive loss of physiological and cellular functions, but the molecular basis of this decline remains unclear. We explored how aging affects transcriptional dynamics using single-cell RNA sequencing of unstimulated and stimulated naïve and effector memory CD4+ T cells from young and old mice from two divergent species. In young animals, immunological activation drives a conserved transcriptomic switch, resulting in tightly controlled gene expression characterized by a strong up-regulation of a core activation program, coupled with a decrease in cell-to-cell variability. Aging perturbed the activation of this core program and increased expression heterogeneity across populations of cells in both species. These discoveries suggest that increased cell-to-cell transcriptional variability will be a hallmark feature of aging across most, if not all, mammalian tissues.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">doi10.1126/science.aah4115</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Martinez-Jimenez, Celia Pilar and Eling, Nils and Chen, Hung-Chang and Vallejos, Catalina A. and Kolodziejczyk, Aleksandra A. and Connor, Frances and Stojic, Lovorka and Rayner, Timothy F. and Stubbington, Michael J. T. and Teichmann, Sarah A. and de la Roche, Maike and Marioni, John C. and Odom, Duncan T.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Aging increases cell-to-cell transcriptional variability upon immune stimulation}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{355}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6332}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1433-1436}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.aah4115}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.science.org/doi/abs/10.1126/science.aah4115}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.science.org/doi/pdf/10.1126/science.aah4115}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2016</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">RSS A</abbr></div> <div id="Vallejos_2016" class="col-sm-8"> <div class="title">Bayesian survival modelling of university outcomes</div> <div class="author"> <em>Catalina A. Vallejos</em>, and <a href="https://warwick.ac.uk/fac/sci/statistics/staff/academic-research/steel/steel_homepage" target="_blank" rel="noopener noreferrer">Mark F. J. Steel</a> </div> <div class="periodical"> <em>Journal of the Royal Statistical Society: Series A (Statistics in Society)</em> Jul 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1111/rssa.12211" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p> Dropouts and delayed graduations are critical issues in higher education systems world wide. A key task in this context is to identify risk factors associated with these events, providing potential targets for mitigating policies. For this, we employ a discrete time competing risks survival model, dealing simultaneously with university outcomes and its associated temporal component. We define survival times as the duration of the student’s enrolment at university and possible outcomes as graduation or two types of dropout (voluntary and involuntary), exploring the information recorded at admission time (e.g. educational level of the parents) as potential predictors. Although similar strategies have been previously implemented, we extend the previous methods by handling covariate selection within a Bayesian variable selection framework, where model uncertainty is formally addressed through Bayesian model averaging. Our methodology is general; however, here we focus on undergraduate students enrolled in three selected degree programmes of the Pontificia Universidad Católica de Chile during the period 2000–2011. Our analysis reveals interesting insights, highlighting the main covariates that influence students’ risk of dropout and delayed graduation.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos_2016</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{RSS A}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A. and Steel, Mark F. J.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of the Royal Statistical Society: Series A (Statistics in Society)}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Bayesian survival modelling of university outcomes}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{613--631}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{180}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/rssa.12211}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1111/rssa.12211}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Vallejos2018" class="col-sm-8"> <div class="title">Beyond comparisons of means: understanding changes in gene expression at the single-cell level</div> <div class="author"> <em>Catalina A. Vallejos</em>, Sylvia Richardson, and <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C. Marioni</a> </div> <div class="periodical"> <em>Genome Biology</em> Jul 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-016-0930-3" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Traditional differential expression tools are limited to detecting changes in overall expression, and fail to uncover the rich information provided by single-cell level data sets. We present a Bayesian hierarchical model that builds upon BASiCS to study changes that lie beyond comparisons of means, incorporating built-in normalization and quantifying technical artifacts by borrowing information from spike-in genes. Using a probabilistic approach, we highlight genes undergoing changes in cell-to-cell heterogeneity but whose overall expression remains unchanged. Control experiments validate our method’s performance and a case study suggests that novel biological insights can be revealed. Our method is implemented in R and available at https://github.com/catavallejos/BASiCS.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2018</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A. and Richardson, Sylvia and Marioni, John C.}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2016/04/15}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-016-0930-3}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Vallejos2016}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{70}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Beyond comparisons of means: understanding changes in gene expression at the single-cell level}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-016-0930-3}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-016-0930-3}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2015</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PLOS</abbr></div> <div id="Vallejos2015" class="col-sm-8"> <div class="title">BASiCS: Bayesian Analysis of Single-Cell Sequencing Data</div> <div class="author">Vallejos Catalina A, Marioni John C, and Richardson Sylvia </div> <div class="periodical"> <em>PLOS Computational Biology</em> Jun 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1371/journal.pcbi.1004333" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell mRNA sequencing can uncover novel cell-to-cell heterogeneity in gene expression levels in seemingly homogeneous populations of cells. However, these experiments are prone to high levels of unexplained technical noise, creating new challenges for identifying genes that show genuine heterogeneous expression within the population of cells under study. BASiCS (Bayesian Analysis of Single-Cell Sequencing data) is an integrated Bayesian hierarchical model where: (i) cell-specific normalisation constants are estimated as part of the model parameters, (ii) technical variability is quantified based on spike-in genes that are artificially introduced to each analysed cell’s lysate and (iii) the total variability of the expression counts is decomposed into technical and biological components. BASiCS also provides an intuitive detection criterion for highly (or lowly) variable genes within the population of cells under study. This is formalised by means of tail posterior probabilities associated to high (or low) biological cell-to-cell variance contributions, quantities that can be easily interpreted by users. We demonstrate our method using gene expression measurements from mouse Embryonic Stem Cells. Cross-validation and meaningful enrichment of gene ontology categories within genes classified as highly (or lowly) variable supports the efficacy of our approach.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2015</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{PLOS}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1004333}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{A, Vallejos Catalina and C, Marioni John and Sylvia, Richardson}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Computational Biology}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{BASiCS: Bayesian Analysis of Single-Cell Sequencing Data}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jun</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1371/journal.pcbi.1004333}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-18}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JASS</abbr></div> <div id="Vallejos2016" class="col-sm-8"> <div class="title">Objective Bayesian Survival Analysis Using Shape Mixtures of Log-Normal Distributions</div> <div class="author"> <em>Catalina A Vallejos</em>, and Mark FJ Steel </div> <div class="periodical"> <em>Journal of the American Statistical Association</em> Jun 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1080/01621459.2014.923316" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Survival models such as the Weibull or log-normal lead to inference that is not robust to the presence of outliers. They also assume that all heterogeneity between individuals can be modeled through covariates. This article considers the use of infinite mixtures of lifetime distributions as a solution for these two issues. This can be interpreted as the introduction of a random effect in the survival distribution. We introduce the family of shape mixtures of log-normal distributions, which covers a wide range of density and hazard functions. Bayesian inference under nonsubjective priors based on the Jeffreys’ rule is examined and conditions for posterior propriety are established. The existence of the posterior distribution on the basis of a sample of point observations is not always guaranteed and a solution through set observations is implemented. In addition, we propose a method for outlier detection based on the mixture structure. A simulation study illustrates the performance of our methods under different scenarios and an application to a real dataset is provided. Supplementary materials for the article, which include R code, are available online.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2016</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{JASS}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A and Steel, Mark FJ}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Objective Bayesian Survival Analysis Using Shape Mixtures of Log-Normal Distributions}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of the American Statistical Association}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{110}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{510}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{697-710}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Taylor &amp; Francis}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1080/01621459.2014.923316}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1080/01621459.2014.923316}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1080/01621459.2014.923316}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0" style="text-align: right;"> © Copyright 2025 </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script src="/assets/js/zoom.js"></script> <script src="/assets/js/common.js"></script> <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </div> </body> </html>
