<!DOCTYPE html> <html lang="en" class=> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> | Publications</title> <meta name="author" content=" "/> <meta name="description" content="The Vallejos Research Group Website "/> <meta name="keywords" content="Bayesian Single-cell RNA-seq EHR"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/favicon.png"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://vallejosgroup.github.io/publications/"> <link rel="apple-touch-icon" href="https://vallejosgroup.github.io/assets/img/favicon.png"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <div class="font-sans"> <header> <nav id="navbar" class=" navbar font-sans font-weight-lighter bg-opacity-30 bg-slate-900 backdrop-filter border-b-2 border-gray-400 border-transparent backdrop-blur-sm navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="https://vallejosgroup.github.io/"><picture> <source srcset="/assets/img/logo-480.webp" type="image/webp"> <source srcset="/assets/img/logo.png" type="image/png"> <img src="/assets/img/logo" alt="" width="100" height="56"> </source></source></picture></a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link navbarcolor" href="/"> About</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/people/">People</a> </li> <li class="nav-item active"> <a class="nav-link navbarcolor" href="/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/software/">Software</a> </li> <li class="nav-item "> <a class="nav-link navbarcolor" href="/contact/">Contact Us</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="text-2xl grow font-sans font-extralight">Publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="year">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JACC</abbr></div> <div id="doi:10.1016/j.jacc.2022.10.025" class="col-sm-8"> <div class="title">Improving Risk Stratification for Patients With Type 2 Myocardial Infarction</div> <div class="author">Caelan Taggart,  <em>Karla Monterrubio-Gómez</em>, Andreas Roos, Jasper Boeddinghaus, Dorien M. Kimenai, Erik Kadesjo, Anda Bularga, Ryan Wereski, Amy Ferry, Matthew Lowry, Atul Anand, Kuan Ken Lee, Dimitrios Doudesis, Ioanna Manolopoulou, Thomas Nestelberger, Luca Koechlin, Pedro Lopez-Ayala, Christian Mueller, Nicholas L. Mills,  <em>Catalina A. Vallejos</em>, and Andrew R. Chapman </div> <div class="periodical"> <em>Journal of the American College of Cardiology</em> Jan 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.jacc.org/doi/abs/10.1016/j.jacc.2022.10.025" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background: Despite poor cardiovascular outcomes, there are no dedicated, validated risk stratification tools to guide investigation or treatment in type 2 myocardial infarction. Objectives: The goal of this study was to derive and validate a risk stratification tool for the prediction of death or future myocardial infarction in patients with type 2 myocardial infarction. Methods: The T2-risk score was developed in a prospective multicenter cohort of consecutive patients with type 2 myocardial infarction. Cox proportional hazards models were constructed for the primary outcome of myocardial infarction or death at 1 year using variables selected a priori based on clinical importance. Discrimination was assessed by area under the receiving-operating characteristic curve (AUC). Calibration was investigated graphically. The tool was validated in a single-center cohort of consecutive patients and in a multicenter cohort study from sites across Europe. Results: There were 1,121, 250, and 253 patients in the derivation, single-center, and multicenter validation cohorts, with the primary outcome occurring in 27% (297 of 1,121), 26% (66 of 250), and 14% (35 of 253) of patients, respectively. The T2-risk score incorporating age, ischemic heart disease, heart failure, diabetes mellitus, myocardial ischemia on electrocardiogram, heart rate, anemia, estimated glomerular filtration rate, and maximal cardiac troponin concentration had good discrimination (AUC: 0.76; 95% CI: 0.73-0.79) for the primary outcome and was well calibrated. Discrimination was similar in the consecutive patient (AUC: 0.83; 95% CI: 0.77-0.88) and multicenter (AUC: 0.74; 95% CI: 0.64-0.83) cohorts. T2-risk provided improved discrimination over the Global Registry of Acute Coronary Events 2.0 risk score in all cohorts. Conclusions: The T2-risk score performed well in different health care settings and could help clinicians to prognosticate, as well as target investigation and preventative therapies more effectively. (High-Sensitivity Troponin in the Evaluation of Patients With Suspected Acute Coronary Syndrome [High-STEACS]; NCT01852123)</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">doi:10.1016/j.jacc.2022.10.025</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Taggart, Caelan and Monterrubio-Gómez, Karla and Roos, Andreas and Boeddinghaus, Jasper and Kimenai, Dorien M. and Kadesjo, Erik and Bularga, Anda and Wereski, Ryan and Ferry, Amy and Lowry, Matthew and Anand, Atul and Lee, Kuan Ken and Doudesis, Dimitrios and Manolopoulou, Ioanna and Nestelberger, Thomas and Koechlin, Luca and Lopez-Ayala, Pedro and Mueller, Christian and Mills, Nicholas L. and Vallejos, Catalina A. and Chapman, Andrew R.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Improving Risk Stratification for Patients With Type 2 Myocardial Infarction}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of the American College of Cardiology}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{JACC}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{81}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{156-168}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.jacc.2022.10.025}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.jacc.org/doi/abs/10.1016/j.jacc.2022.10.025}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.jacc.org/doi/pdf/10.1016/j.jacc.2022.10.025}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="year">2022</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">medRxiv</abbr></div> <div id="ConstantineCooke2022" class="col-sm-8"> <div class="title">Latent Crohn’s Disease Subgroups are Identified by Longitudinal Faecal Calprotectin Profiles</div> <div class="author"> <em>Nathan Samuel Constantine-Cooke</em>,  <em>Karla Monterrubio Gomez</em>, Nikolas Plevris, Lauranne A.A.P Derikx, Beatriz Gros, Gareth-Rhys Jones, <a href="https://marioni-group.owlstown.net" target="_blank" rel="noopener noreferrer">Riccardo Marioni</a>, <a href="https://charlielees.com" target="_blank" rel="noopener noreferrer">Charlie W. Lees</a>, and <em>Catalina Vallejos</em> </div> <div class="periodical"> <em></em> Aug 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.medrxiv.org/content/10.1101/2022.08.16.22278320v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> <a href="/assets/pdf/LCMM-Preprint.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Background High faecal calprotectin is associated with poor outcomes in Crohn’s disease. Monitoring of faecal calprotectin trajectories could characterise disease progression before severe complications occur. Aims We undertook an unbiased assessment of a retrospective incident Crohn’s disease cohort to assess for inter-individual variability in faecal calprotectin levels over time. We aimed to explore whether latent classes of such profiles are associated with a composite endpoint consisting of surgery, hospitalisation, or Montreal behaviour progression and other clinical information. Methods Latent class mixed models were used to model faecal calprotectin trajectories within five years of diagnosis. Akaike information criterion, Bayesian information criterion, alluvial plots, and class-specific trajectories were used to decide the optimal number of classes. Log-rank tests of Kaplan-Meier estimators were used to test for associations between class membership and outcomes. Results Our study cohort comprised 365 subjects and 2856 faecal calprotectin measurements (median 7 per subject). Four latent classes were found and broadly described as a class with consistently high faecal calprotectin and three classes characterised by downward trends for calprotectin. Class membership was significantly associated with the composite endpoint, and separately, hospitalisation and Montreal disease progression, but not surgery. Early biologic therapy was strongly associated with class membership. Conclusions Our analysis provides a novel stratification approach for Crohn’s disease patients based on faecal calprotectin trajectories. Characterising this heterogeneity helps to better understand different patterns of disease progression and to identify those with a higher risk of worse outcomes. Ultimately, this information will assist the design of more targeted interventions.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">ConstantineCooke2022</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{LCMM-Preprint.pdf}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Constantine-Cooke, Nathan Samuel and Gomez, Karla Monterrubio and Plevris, Nikolas and Derikx, Lauranne A.A.P and Gros, Beatriz and Jones, Gareth-Rhys and Marioni, Riccardo and Lees, Charlie W. and Vallejos, Catalina}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Latent Crohn{\textquotesingle}s Disease Subgroups are Identified by Longitudinal Faecal Calprotectin Profiles}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2022.08.16.22278320}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/10.1101/2022.08.16.22278320v1}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="Monterrubio2020" class="col-sm-8"> <div class="title">A review on competing risks methods for survival analysis</div> <div class="author"> <em>Karla Monterrubio-Gómez</em>,  <em>Nathan Constantine-Cooke</em>, and <em>Catalina A Vallejos</em> </div> <div class="periodical"> <em>arXiv</em> Dec 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://arxiv.org/abs/2212.05157" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>When modelling competing risks survival data, several techniques have been proposed in both the statistical and machine learning literature. State-of-the-art methods have extended classical approaches with more flexible assumptions that can improve predictive performance, allow high dimensional data and missing values, among others. Despite this, modern approaches have not been widely employed in applied settings. This article aims to aid the uptake of such methods by providing a condensed compendium of competing risks survival methods with a unified notation and interpretation across approaches. We highlight available software and, when possible, demonstrate their usage via reproducible R vignettes. Moreover, we discuss two major concerns that can affect benchmark studies in this context: the choice of performance metrics and reproducibility.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Monterrubio2020</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Monterrubio-G{\'o}mez, Karla and Constantine-Cooke, Nathan and Vallejos, Catalina A}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A review on competing risks methods for survival analysis}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.48550/ARXIV.2212.05157}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://arxiv.org/abs/2212.05157}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2021</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Kapourani2021" class="col-sm-8"> <div class="title">scMET: Bayesian modeling of DNA methylation heterogeneity at single-cell resolution</div> <div class="author"> <em>Chantriolnt-Andreas Kapourani</em>, Ricard Argelaguet, <a href="https://datascience.sissa.it/person/11/guido-sanguinetti" target="_blank" rel="noopener noreferrer">Guido Sanguinetti</a>, and <em>Catalina A Vallejos</em> </div> <div class="periodical"> <em>Genome Biology</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-021-02329-8" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>High-throughput single-cell measurements of DNA methylomes can quantify methylation heterogeneity and uncover its role in gene regulation. However, technical limitations and sparse coverage can preclude this task. scMET is a hierarchical Bayesian model which overcomes sparsity, sharing information across cells and genomic features to robustly quantify genuine biological heterogeneity. scMET can identify highly variable features that drive epigenetic heterogeneity, and perform differential methylation and variability analyses. We illustrate how scMET facilitates the characterization of epigenetically distinct cell populations and how it enables the formulation of novel hypotheses on the epigenetic regulation of gene expression. scMET is available at https://github.com/andreaskapou/scMET.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Kapourani2021</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kapourani, Chantriolnt-Andreas and Argelaguet, Ricard and Sanguinetti, Guido and Vallejos, Catalina A}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2021/04/20}</span><span class="p">,</span>
  <span class="na">date-added</span> <span class="p">=</span> <span class="s">{2022-02-26 12:43:46 +0000}</span><span class="p">,</span>
  <span class="na">date-modified</span> <span class="p">=</span> <span class="s">{2022-02-26 12:43:46 +0000}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Kapourani2021}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{114}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{scMET: Bayesian modeling of DNA methylation heterogeneity at single-cell resolution}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-021-02329-8}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">medRxiv</abbr></div> <div id="Cheng2021" class="col-sm-8"> <div class="title">DNA Methylation scores augment 10-year risk prediction of diabetes</div> <div class="author"> <em>Yipeng Cheng</em>, Danni A Gadd, Christian Gieger,  <em>Karla Monterrubio-Gómez</em>, Yufei Zhang, Imrich Berta, Michael J Stam, Natalia Szlachetka, Evgenii Lobzaev, Archie Campbell, Cliff Nangle, Rosie M Walker, Chloe Fawns-Ritchie, Annette Peters, Wolfgang Rathmann, David J Porteous, Kathryn L Evans, Andrew M McIntosh, Timothy I Cannings, Melanie Waldenberger, Andrea Ganna, Daniel L McCartney,  <em>Catalina A Vallejos</em>, and <a href="https://marioni-group.owlstown.net" target="_blank" rel="noopener noreferrer">Riccardo E Marioni</a> </div> <div class="periodical"> <em>medRxiv</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.medrxiv.org/content/early/2021/11/21/2021.11.19.21266469" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> <a href="/assets/pdf/cheng2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Type 2 diabetes mellitus (T2D) is one of the most prevalent diseases in the world and presents a major health and economic burden, a notable proportion of which could be alleviated with improved early prediction and intervention. While standard risk factors including age, obesity, and hypertension have shown good predictive performance, we show that the use of CpG DNA methylation information leads to a significant improvement in the prediction of 10-year T2D incidence risk. Whilst previous studies have been largely constrained by linear assumptions and the use of CpGs one-at-the-time, we have adopted a more flexible approach based on a range of linear and tree-ensemble models for classification and time-to-event prediction. Using the Generation Scotland cohort (n=9,537) our best performing model (Area Under the Curve (AUC)=0.880, Precision Recall AUC (PRAUC)=0.539, McFadden’s R2=0.316) used a LASSO Cox proportional-hazards predictor and showed notable improvement in onset prediction, above and beyond standard risk factors (AUC=0.860, PRAUC=0.444 R2=0.261). Replication of the main finding was observed in an external test dataset (the German-based KORA study, p=3.7x10-4). Tree-ensemble methods provided comparable performance and future improvements to these models are discussed. Finally, we introduce MethylPipeR, an R package with accompanying user interface, for systematic and reproducible development of complex trait and incident disease predictors. While MethylPipeR was applied to incident T2D prediction with DNA methylation in our experiments, the package is designed for generalised development of predictive models and is applicable to a wide range of omics data and target traits.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Cheng2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Cheng, Yipeng and Gadd, Danni A and Gieger, Christian and Monterrubio-Gómez, Karla and Zhang, Yufei and Berta, Imrich and Stam, Michael J and Szlachetka, Natalia and Lobzaev, Evgenii and Campbell, Archie and Nangle, Cliff and Walker, Rosie M and Fawns-Ritchie, Chloe and Peters, Annette and Rathmann, Wolfgang and Porteous, David J and Evans, Kathryn L and McIntosh, Andrew M and Cannings, Timothy I and Waldenberger, Melanie and Ganna, Andrea and McCartney, Daniel L and Vallejos, Catalina A and Marioni, Riccardo E}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{DNA Methylation scores augment 10-year risk prediction of diabetes}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2021.11.19.21266469}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2021.11.19.21266469}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory Press}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2021/11/21/2021.11.19.21266469}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{cheng2021.pdf}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="liley2021" class="col-sm-8"> <div class="title">Model updating after interventions paradoxically introduces bias</div> <div class="author"> <em>James Liley</em>, Samuel R Emerson, Bilal A Mateen,  <em>Catalina A Vallejos</em>, Louis J M Aslett, and Sebastian J Vollmer </div> <div class="periodical"> <em>arXiv</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2010.11530" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/liley2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract secret"> <p>Machine learning is increasingly being used to generate prediction models for use in a number of real-world settings, from credit risk assessment to clinical decision support. Recent discussions have highlighted potential problems in the updating of a predictive score for a binary outcome when an existing predictive score forms part of the standard workflow, driving interventions. In this setting, the existing score induces an additional causative pathway which leads to miscalibration when the original score is replaced. We propose a general causal framework to describe and address this problem, and demonstrate an equivalent formulation as a partially observed Markov decision process. We use this model to demonstrate the impact of such ‘naive updating’ when performed repeatedly. Namely, we show that successive predictive scores may converge to a point where they predict their own effect, or may eventually tend toward a stable oscillation between two values, and we argue that neither outcome is desirable. Furthermore, we demonstrate that even if model-fitting procedures improve, actual performance may worsen. We complement these findings with a discussion of several potential routes to overcome these issues.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liley2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Model updating after interventions paradoxically introduces bias}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{liley2021.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liley, James and Emerson, Samuel R and Mateen, Bilal A and Vallejos, Catalina A and Aslett, Louis J M and Vollmer, Sebastian J}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">arxiv</span> <span class="p">=</span> <span class="s">{2010.11530}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">primaryclass</span> <span class="p">=</span> <span class="s">{stat.ML}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NatCom</abbr></div> <div id="Richter2021" class="col-sm-8"> <div class="title">Single-nucleus RNA-seq2 reveals functional crosstalk between liver zonation and ploidy</div> <div class="author">M. L. Richter, I. K. Deligiannis, K. Yin, A. Danese, E. Lleshi, P. Coupland,  <em>C. A. Vallejos</em>, K. P. Matchett, N. C. Henderson, M. Colome-Tatche, and C. P. Martinez-Jimenez </div> <div class="periodical"> <em>Nature Communications</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038/s41467-021-24543-5" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell RNA-seq reveals the role of pathogenic cell populations in development and progression of chronic diseases. In order to expand our knowledge on cellular heterogeneity, we have developed a single-nucleus RNA-seq2 method tailored for the comprehensive analysis of the nuclear transcriptome from frozen tissues, allowing the dissection of all cell types present in the liver, regardless of cell size or cellular fragility. We use this approach to characterize the transcriptional profile of individual hepatocytes with different levels of ploidy, and have discovered that ploidy states are associated with different metabolic potential, and gene expression in tetraploid mononucleated hepatocytes is conditioned by their position within the hepatic lobule. Our work reveals a remarkable crosstalk between gene dosage and spatial distribution of hepatocytes.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Richter2021</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NatCom}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Richter, M. L. and Deligiannis, I. K. and Yin, K. and Danese, A. and Lleshi, E. and Coupland, P. and Vallejos, C. A. and Matchett, K. P. and Henderson, N. C. and Colome-Tatche, M. and Martinez-Jimenez, C. P.}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2021/07/12}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-021-24543-5}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Richter2021}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{2041-1723}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{4264}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Single-nucleus RNA-seq2 reveals functional crosstalk between liver zonation and ploidy}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/s41467-021-24543-5}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">medRxiv</abbr></div> <div id="Liley2021.08.06.21261593" class="col-sm-8"> <div class="title">Development and assessment of a machine learning tool for predicting emergency admission in Scotland</div> <div class="author"> <em>James Liley</em>, Gergo Bohner, Samuel R. Emerson, Bilal A. Mateen, Katie Borland, David Carr, Scott Heald, Samuel D. Oduro, Jill Ireland, Keith Moffat, Rachel Porteous, Stephen Riddell, Nathan Cunningham, Chris Holmes, Katrina Payne, Sebastian J. Vollmer,  <em>Catalina A. Vallejos</em>, and Louis J. M. Aslett </div> <div class="periodical"> <em>medRxiv</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.medrxiv.org/content/early/2021/08/10/2021.08.06.21261593" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Avoiding emergency hospital admission (EA) is advantageous to individual health and the healthcare system. We develop a statistical model estimating risk of EA for most of the Scottish population (&gt; 4.8M individuals) using electronic health records, such as hospital episodes and prescribing activity. We demonstrate good predictive accuracy (AUROC 0.80), calibration and temporal stability. We find strong prediction of respiratory and metabolic EA, show a substantial risk contribution from socioeconomic decile, and highlight an important problem in model updating. Our work constitutes a rare example of a population-scale machine learning score to be deployed in a healthcare setting.Competing Interest StatementThe authors have declared no competing interest.Funding StatementJL, CAV and LJMA were partially supported by Wave 1 of The UKRI Strategic Priorities Fund under the EPSRC Grant EP/T001569/1, particularly the "Health" theme within that grant and The Alan Turing Institute; JL, BAM, CAV, LJMA and SJV were partially supported by Health Data Research UK, an initiative funded by UK Research and Innovation, Department of Health and Social Care (England), the devolved administrations, and leading medical research charities; SJV, NC and GB were partially supported by the University of Warwick Impact Fund. SRE is funded by the EPSRC doctoral training partnership (DTP) at Durham University, grant reference EP/R513039/1; LJMA was partially supported by a Health Programme Fellowship at The Alan Turing Institute; CAV was supported by a Chancellor’s Fellowship provided by the University of Edinburgh.Author DeclarationsI confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.YesThe details of the IRB/oversight body that provided approval or exemption for the research described are given below:This study and the use of NHS data was approved by the Public Benefit and Privacy Panel for Health and Social Care (study number 1718-0370; approval evidenced in application outcome minutes for 2018/19 at https://www.informationgovernance.scot.nhs.uk/pbpphsc/application-outcomes/ ). In addition, accessing data was approved by the Public Health Scotland National Safe Haven, through the the electronic Data Research and Innovation Service (eDRIS) and the Public Benefit and Privacy Panel (PBPP) (study number 1718-0370). All studies have been conducted in accordance with information governance standards; data had no patient identifiers available to the researchers. This work was conducted in accordance with UK data governance regulations under PBPP application number eDRIS 1718-0370 All necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived.YesI understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).YesI have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.YesRaw data for this project are patient-level NHS Scotland health records, and are confidential. Due to the confidential nature of the data used, all analysis took place on remote ’safe havens’, without access to internet, software updates or unpublished software. Information Governance training was required for all researchers accessing the analysis environment. Moreover, to avoid the risk of accidental disclosure of sensitive information, an independent team carried out statistical disclosure control checks to all data exports, including the outputs presented in this manuscript. All analysis code and co-ordinates required to reproduce our Figures are available in github.com/jamesliley/SPARRAv4 https://github.com/jamesliley/SPARRAv4</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Liley2021.08.06.21261593</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liley, James and Bohner, Gergo and Emerson, Samuel R. and Mateen, Bilal A. and Borland, Katie and Carr, David and Heald, Scott and Oduro, Samuel D. and Ireland, Jill and Moffat, Keith and Porteous, Rachel and Riddell, Stephen and Cunningham, Nathan and Holmes, Chris and Payne, Katrina and Vollmer, Sebastian J. and Vallejos, Catalina A. and Aslett, Louis J. M.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Development and assessment of a machine learning tool for predicting emergency admission in Scotland}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2021.08.06.21261593}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2021.08.06.21261593}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory Press}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2021/08/10/2021.08.06.21261593}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.medrxiv.org/content/early/2021/08/10/2021.08.06.21261593.full.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{medRxiv}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div> <div id="Maniatis2021.05.13.443959" class="col-sm-8"> <div class="title">SCRaPL: hierarchical Bayesian modelling of associations in single cell multi-omics data</div> <div class="author"> <em>Christos Maniatis</em>,  <em>Catalina A Vallejos</em>, and <a href="https://datascience.sissa.it/person/11/guido-sanguinetti" target="_blank" rel="noopener noreferrer">Guido Sanguinetti</a> </div> <div class="periodical"> <em>bioRxiv</em> Dec 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/early/2021/05/14/2021.05.13.443959" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell multi-omics assays offer unprecedented opportunities to explore gene regulation at cellular level. However, high levels of technical noise and data sparsity frequently lead to a lack of statistical power in correlative analyses, identifying very few, if any, significant associations between different molecular layers. Here we propose SCRaPL, a novel computational tool that increases power by carefully modelling noise in the experimental systems. We show on real and simulated multi-omics single-cell data sets that SCRaPL achieves higher sensitivity and better robustness in identifying correlations, while maintaining a similar level of false positives as standard analyses based on Pearson correlation.Competing Interest StatementThe authors have declared no competing interest.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Maniatis2021.05.13.443959</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Maniatis, Christos and Vallejos, Catalina A and Sanguinetti, Guido}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SCRaPL: hierarchical Bayesian modelling of associations in single cell multi-omics data}</span><span class="p">,</span>
  <span class="na">elocation-id</span> <span class="p">=</span> <span class="s">{2021.05.13.443959}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2021.05.13.443959}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cold Spring Harbor Laboratory}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/early/2021/05/14/2021.05.13.443959}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/early/2021/05/14/2021.05.13.443959.full.pdf}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2020</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Lahnemann2020" class="col-sm-8"> <div class="title">Eleven grand challenges in single-cell data science</div> <div class="author">David Lähnemann, Johannes Köster, Ewa Szczurek, Davis J. McCarthy, Stephanie C. Hicks, Mark D. Robinson,  <em>Catalina A. Vallejos</em>, Kieran R. Campbell, Niko Beerenwinkel, Ahmed Mahfouz, Luca Pinello, Pavel Skums, Alexandros Stamatakis, Camille Stephan-Otto Attolini, Samuel Aparicio, Jasmijn Baaijens, Marleen Balvert, Buys de Barbanson, Antonio Cappuccio, Giacomo Corleone, Bas E. Dutilh, Maria Florescu, Victor Guryev, Rens Holmer, Katharina Jahn, Thamar Jessurun Lobo, Emma M. Keizer, Indu Khatri, Szymon M. Kielbasa, Jan O. Korbel, Alexey M. Kozlov, Tzu-Hao Kuo, Boudewijn P. F. Lelieveldt, Ion I. Mandoiu, <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C. Marioni</a>, Tobias Marschall, Felix Mölder, Amir Niknejad, Lukasz Raczkowski, Marcel Reinders, Jeroen de Ridder, Antoine-Emmanuel Saliba, Antonios Somarakis, Oliver Stegle, Fabian J. Theis, Huan Yang, Alex Zelikovsky, Alice C. McHardy, Benjamin J. Raphael, Sohrab P. Shah, and Alexander Schönhuth </div> <div class="periodical"> <em>Genome Biology</em> Dec 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-020-1926-6" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>The recent boom in microfluidics and combinatorial indexing strategies, combined with low sequencing costs, has empowered single-cell sequencing technology. Thousands—or even millions—of cells analyzed in a single experiment amount to a data revolution in single-cell biology and pose unique data science problems. Here, we outline eleven challenges that will be central to bringing this emerging field of single-cell data science forward. For each challenge, we highlight motivating research questions, review prior work, and formulate open problems. This compendium is for established researchers, newcomers, and students alike, highlighting interesting and rewarding problems for the coming years.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Lahnemann2020</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{L{\"a}hnemann, David and K{\"o}ster, Johannes and Szczurek, Ewa and McCarthy, Davis J. and Hicks, Stephanie C. and Robinson, Mark D. and Vallejos, Catalina A. and Campbell, Kieran R. and Beerenwinkel, Niko and Mahfouz, Ahmed and Pinello, Luca and Skums, Pavel and Stamatakis, Alexandros and Attolini, Camille Stephan-Otto and Aparicio, Samuel and Baaijens, Jasmijn and Balvert, Marleen and Barbanson, Buys de and Cappuccio, Antonio and Corleone, Giacomo and Dutilh, Bas E. and Florescu, Maria and Guryev, Victor and Holmer, Rens and Jahn, Katharina and Lobo, Thamar Jessurun and Keizer, Emma M. and Khatri, Indu and Kielbasa, Szymon M. and Korbel, Jan O. and Kozlov, Alexey M. and Kuo, Tzu-Hao and Lelieveldt, Boudewijn P. F. and Mandoiu, Ion I. and Marioni, John C. and Marschall, Tobias and M{\"o}lder, Felix and Niknejad, Amir and Raczkowski, Lukasz and Reinders, Marcel and Ridder, Jeroen de and Saliba, Antoine-Emmanuel and Somarakis, Antonios and Stegle, Oliver and Theis, Fabian J. and Yang, Huan and Zelikovsky, Alex and McHardy, Alice C. and Raphael, Benjamin J. and Shah, Sohrab P. and Sch{\"o}nhuth, Alexander}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2020/02/07}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-020-1926-6}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{L{\"a}hnemann2020}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{31}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Eleven grand challenges in single-cell data science}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-020-1926-6}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-020-1926-6}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Circ</abbr></div> <div id="doi:10.1161/CIRCULATIONAHA.119.042960" class="col-sm-8"> <div class="title">High-Sensitivity Cardiac Troponin and the Universal Definition of Myocardial Infarction</div> <div class="author">Andrew R. Chapman, Philip D. Adamson, Anoop S.V. Shah, Atul Anand, Fiona E. Strachan, Amy V. Ferry, Kuan Ken Lee, Colin Berry, Iain Findlay, Anne Cruikshank, Alan Reid, Alasdair Gray, Paul O. Collinson, Fred Apple, David A. McAllister, Donogh Maguire, Keith A.A. Fox,  <em>Catalina A. Vallejos</em>, Catriona Keerie, Christopher J. Weir, David E. Newby, Nicholas L. Mills, Christopher Tuck, Anda Bularga, Ryan Wereski, Dennis Sandeman, Catherine L. Stables, Athanasios Tsanasis, Lucy Marshall, Stacey D. Stewart, Takeshi Fujisawa, Mischa Hautvast, Jean McPherson, Lynn McKinlay, Simon Walker, Ian Ford, Simon Walker, Shannon Amoils, Jennifer Stevens, John Norrie, Jack Andrews, Phil Adamson, Alastair Moss, Mohamed Anwar, John Hung, Simon Walker, Jonathan Malo, Colin Fischbacher, Bernard Croal, Stephen J. Leslie, Richard Parker, Allan Walker, Ronnie Harkess, Chris Tuck, Tony Wackett, Roma Armstrong, Marion Flood, Laura Stirling, Claire MacDonald, Imran Sadat, Frank Finlay, Heather Charles, Pamela Linksted, Stephen Young, Bill Alexander, and Chris Duncan </div> <div class="periodical"> <em>Circulation</em> Dec 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.ahajournals.org/doi/abs/10.1161/CIRCULATIONAHA.119.042960" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background: The introduction of more sensitive cardiac troponin assays has led to increased recognition of myocardial injury in acute illnesses other than acute coronary syndrome. The Universal Definition of Myocardial Infarction recommends high-sensitivity cardiac troponin testing and classification of patients with myocardial injury based on pathogenesis, but the clinical implications of implementing this guideline are not well understood. Methods: In a stepped-wedge cluster randomized, controlled trial, we implemented a high-sensitivity cardiac troponin assay and the recommendations of the Universal Definition in 48 282 consecutive patients with suspected acute coronary syndrome. In a prespecified secondary analysis, we compared the primary outcome of myocardial infarction or cardiovascular death and secondary outcome of noncardiovascular death at 1 year across diagnostic categories. Results: Implementation increased the diagnosis of type 1 myocardial infarction by 11% (510/4471), type 2 myocardial infarction by 22% (205/916), and acute and chronic myocardial injury by 36% (443/1233) and 43% (389/898), respectively. Compared with those without myocardial injury, the rate of the primary outcome was highest in those with type 1 myocardial infarction (cause-specific hazard ratio [HR] 5.64 [95% CI, 5.12–6.22]), but was similar across diagnostic categories, whereas noncardiovascular deaths were highest in those with acute myocardial injury (cause specific HR 2.65 [95% CI, 2.33–3.01]). Despite modest increases in antiplatelet therapy and coronary revascularization after implementation in patients with type 1 myocardial infarction, the primary outcome was unchanged (cause specific HR 1.00 [95% CI, 0.82–1.21]). Increased recognition of type 2 myocardial infarction and myocardial injury did not lead to changes in investigation, treatment or outcomes. Conclusions: Implementation of high-sensitivity cardiac troponin assays and the recommendations of the Universal Definition of Myocardial Infarction identified patients at high-risk of cardiovascular and noncardiovascular events but was not associated with consistent increases in treatment or improved outcomes. Trials of secondary prevention are urgently required to determine whether this risk is modifiable in patients without type 1 myocardial infarction.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">doi:10.1161/CIRCULATIONAHA.119.042960</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Circ}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chapman, Andrew R. and Adamson, Philip D. and Shah, Anoop S.V. and Anand, Atul and Strachan, Fiona E. and Ferry, Amy V. and Lee, Kuan Ken and Berry, Colin and Findlay, Iain and Cruikshank, Anne and Reid, Alan and Gray, Alasdair and Collinson, Paul O. and Apple, Fred and McAllister, David A. and Maguire, Donogh and Fox, Keith A.A. and Vallejos, Catalina A. and Keerie, Catriona and Weir, Christopher J. and Newby, David E. and Mills, Nicholas L. and Tuck, Christopher and Bularga, Anda and Wereski, Ryan and Sandeman, Dennis and Stables, Catherine L. and Tsanasis, Athanasios and Marshall, Lucy and Stewart, Stacey D. and Fujisawa, Takeshi and Hautvast, Mischa and McPherson, Jean and McKinlay, Lynn and Walker, Simon and Ford, Ian and Walker, Simon and Amoils, Shannon and Stevens, Jennifer and Norrie, John and Andrews, Jack and Adamson, Phil and Moss, Alastair and Anwar, Mohamed and Hung, John and Walker, Simon and Malo, Jonathan and Fischbacher, Colin and Croal, Bernard and Leslie, Stephen J. and Parker, Richard and Walker, Allan and Harkess, Ronnie and Tuck, Chris and Wackett, Tony and Armstrong, Roma and Flood, Marion and Stirling, Laura and MacDonald, Claire and Sadat, Imran and Finlay, Frank and Charles, Heather and Linksted, Pamela and Young, Stephen and Alexander, Bill and Duncan, Chris}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High-Sensitivity Cardiac Troponin and the Universal Definition of Myocardial Infarction}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Circulation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{141}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{161-171}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1161/CIRCULATIONAHA.119.042960}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.ahajournals.org/doi/abs/10.1161/CIRCULATIONAHA.119.042960}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.ahajournals.org/doi/pdf/10.1161/CIRCULATIONAHA.119.042960}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2018</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CellSys</abbr></div> <div id="Eling2018" class="col-sm-8"> <div class="title">Correcting the Mean-Variance Dependency for Differential Variability Testing Using Single-Cell RNA Sequencing Data</div> <div class="author">Nils Eling, Arianne C Richard, Sylvia Richardson, <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C Marioni</a>, and <em>Catalina A Vallejos</em> </div> <div class="periodical"> <em>Cell Systems</em> Dec 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2405471218302783" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Cell-to-cell transcriptional variability in otherwise homogeneous cell populations plays an important role in tissue function and development. Single-cell RNA sequencing can characterize this variability in a transcriptome-wide manner. However, technical variation and the confounding between variability and mean expression estimates hinder meaningful comparison of expression variability between cell populations. To address this problem, we introduce an analysis approach that extends the BASiCS statistical framework to derive a residual measure of variability that is not confounded by mean expression. This includes a robust procedure for quantifying technical noise in experiments where technical spike-in molecules are not available. We illustrate how our method provides biological insight into the dynamics of cell-to-cell expression variability, highlighting a synchronization of biosynthetic machinery components in immune cells upon activation. In contrast to the uniform up-regulation of the biosynthetic machinery, CD4+ T cells show heterogeneous up-regulation of immune-related and lineage-defining genes during activation and differentiation.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Eling2018</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{CellSys}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Correcting the Mean-Variance Dependency for Differential Variability Testing Using Single-Cell RNA Sequencing Data}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Systems}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{284-294.e12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2405-4712}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.cels.2018.06.011}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S2405471218302783}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Eling, Nils and Richard, Arianne C and Richardson, Sylvia and Marioni, John C and Vallejos, Catalina A}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{single-cell RNA sequencing, transcriptional noise, variability, immune activation, statistics, Bayesian}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Lancet</abbr></div> <div id="SHAH2018919" class="col-sm-8"> <div class="title">High-sensitivity troponin in the evaluation of patients with suspected acute coronary syndrome: a stepped-wedge, cluster-randomised controlled trial</div> <div class="author">Anoop S V Shah, Atul Anand, Fiona E Strachan, Amy V Ferry, Kuan Ken Lee, Andrew R Chapman, Dennis Sandeman, Catherine L Stables, Philip D Adamson, Jack P M Andrews, Mohamed S Anwar, John Hung, Alistair J Moss, Rachel O’Brien, Colin Berry, Iain Findlay, Simon Walker, Anne Cruickshank, Alan Reid, Alasdair Gray, Paul O Collinson, Fred S Apple, David A McAllister, Donogh Maguire, Keith A A Fox, David E Newby, Christopher Tuck, Ronald Harkess, Richard A Parker, Catriona Keerie, Christopher J Weir, Nicholas L Mills, Lucy Marshall, Stacey D Stewart, Takeshi Fujisawa,  <em>Catalina A Vallejos</em>, Athanasios Tsanas, Mischa Hautvast, Jean McPherson, Lynn McKinlay, Jonathan Malo, Colin M Fischbacher, Bernard L Croal, Stephen J Leslie, Allan Walker, Tony Wackett, Roma Armstrong, Laura Stirling, Claire MacDonald, Imran Sadat, Frank Finlay, Heather Charles, Pamela Linksted, Stephen Young, Bill Alexander, and Chris Duncan </div> <div class="periodical"> <em>The Lancet</em> Dec 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0140673618319238" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Background: High-sensitivity cardiac troponin assays permit use of lower thresholds for the diagnosis of myocardial infarction, but whether this improves clinical outcomes is unknown. We aimed to determine whether the introduction of a high-sensitivity cardiac troponin I (hs-cTnI) assay with a sex-specific 99th centile diagnostic threshold would reduce subsequent myocardial infarction or cardiovascular death in patients with suspected acute coronary syndrome. Methods: In this stepped-wedge, cluster-randomised controlled trial across ten secondary or tertiary care hospitals in Scotland, we evaluated the implementation of an hs-cTnI assay in consecutive patients who had been admitted to the hospitals’ emergency departments with suspected acute coronary syndrome. Patients were eligible for inclusion if they presented with suspected acute coronary syndrome and had paired cardiac troponin measurements from the standard care and trial assays. During a validation phase of 6–12 months, results from the hs-cTnI assay were concealed from the attending clinician, and a contemporary cardiac troponin I (cTnI) assay was used to guide care. Hospitals were randomly allocated to early (n=5 hospitals) or late (n=5 hospitals) implementation, in which the high-sensitivity assay and sex-specific 99th centile diagnostic threshold was introduced immediately after the 6-month validation phase or was deferred for a further 6 months. Patients reclassified by the high-sensitivity assay were defined as those with an increased hs-cTnI concentration in whom cTnI concentrations were below the diagnostic threshold on the contemporary assay. The primary outcome was subsequent myocardial infarction or death from cardiovascular causes at 1 year after initial presentation. Outcomes were compared in patients reclassified by the high-sensitivity assay before and after its implementation by use of an adjusted generalised linear mixed model. This trial is registered with ClinicalTrials.gov, number NCT01852123. Findings: Between June 10, 2013, and March 3, 2016, we enrolled 48 282 consecutive patients (61 [SD 17] years, 47% women) of whom 10 360 (21%) patients had cTnI concentrations greater than those of the 99th centile of the normal range of values, who were identified by the contemporary assay or the high-sensitivity assay. The high-sensitivity assay reclassified 1771 (17%) of 10 360 patients with myocardial injury or infarction who were not identified by the contemporary assay. In those reclassified, subsequent myocardial infarction or cardiovascular death within 1 year occurred in 105 (15%) of 720 patients in the validation phase and 131 (12%) of 1051 patients in the implementation phase (adjusted odds ratio for implementation vs validation phase 1·10, 95% CI 0·75 to 1·61; p=0·620). Interpretation: Use of a high-sensitivity assay prompted reclassification of 1771 (17%) of 10 360 patients with myocardial injury or infarction, but was not associated with a lower subsequent incidence of myocardial infarction or cardiovascular death at 1 year. Our findings question whether the diagnostic threshold for myocardial infarction should be based on the 99th centile derived from a normal reference population.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">SHAH2018919</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Lancet}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High-sensitivity troponin in the evaluation of patients with suspected acute coronary syndrome: a stepped-wedge, cluster-randomised controlled trial}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The Lancet}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{392}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10151}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{919-928}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0140-6736}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/S0140-6736(18)31923-8}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S0140673618319238}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Shah, Anoop S V and Anand, Atul and Strachan, Fiona E and Ferry, Amy V and Lee, Kuan Ken and Chapman, Andrew R and Sandeman, Dennis and Stables, Catherine L and Adamson, Philip D and Andrews, Jack P M and Anwar, Mohamed S and Hung, John and Moss, Alistair J and O'Brien, Rachel and Berry, Colin and Findlay, Iain and Walker, Simon and Cruickshank, Anne and Reid, Alan and Gray, Alasdair and Collinson, Paul O and Apple, Fred S and McAllister, David A and Maguire, Donogh and Fox, Keith A A and Newby, David E and Tuck, Christopher and Harkess, Ronald and Parker, Richard A and Keerie, Catriona and Weir, Christopher J and Mills, Nicholas L and Marshall, Lucy and Stewart, Stacey D and Fujisawa, Takeshi and Vallejos, Catalina A and Tsanas, Athanasios and Hautvast, Mischa and McPherson, Jean and McKinlay, Lynn and Malo, Jonathan and Fischbacher, Colin M and Croal, Bernard L and Leslie, Stephen J and Walker, Allan and Wackett, Tony and Armstrong, Roma and Stirling, Laura and MacDonald, Claire and Sadat, Imran and Finlay, Frank and Charles, Heather and Linksted, Pamela and Young, Stephen and Alexander, Bill and Duncan, Chris}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2017</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">NatMet</abbr></div> <div id="Vallejos2017" class="col-sm-8"> <div class="title">Normalizing single-cell RNA sequencing data: challenges and opportunities</div> <div class="author"> <em>Catalina A Vallejos</em>, Davide Risso, Antonio Scialdone, Sandrine Dudoit, and <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C Marioni</a> </div> <div class="periodical"> <em>Nature Methods</em> Dec 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038/nmeth.4292" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell transcriptomics is becoming an important component of the molecular biologist’s toolkit. A critical step when analyzing data generated using this technology is normalization. However, normalization is typically performed using methods developed for bulk RNA sequencing or even microarray data, and the suitability of these methods for single-cell transcriptomics has not been assessed. We here discuss commonly used normalization approaches and illustrate how these can produce misleading results. Finally, we present alternative approaches and provide recommendations for single-cell RNA sequencing users.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2017</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NatMet}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Normalizing single-cell RNA sequencing data: challenges and opportunities}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A and Risso, Davide and Scialdone, Antonio and Dudoit, Sandrine and Marioni, John C}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2017/06/01}</span><span class="p">,</span>
  <span class="na">date-added</span> <span class="p">=</span> <span class="s">{2022-02-26 11:24:03 +0000}</span><span class="p">,</span>
  <span class="na">date-modified</span> <span class="p">=</span> <span class="s">{2022-02-26 11:24:03 +0000}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/nmeth.4292}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1548-7105}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Methods}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{565--571}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/nmeth.4292}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">EconStat</abbr></div> <div id="VALLEJOS201773" class="col-sm-8"> <div class="title">Incorporating unobserved heterogeneity in Weibull survival models: A Bayesian approach</div> <div class="author"> <em>Catalina A. Vallejos</em>, and <a href="https://warwick.ac.uk/fac/sci/statistics/staff/academic-research/steel/steel_homepage" target="_blank" rel="noopener noreferrer">Mark F.J. Steel</a> </div> <div class="periodical"> <em>Econometrics and Statistics</em> Dec 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2452306217300072" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Outlying observations and other forms of unobserved heterogeneity can distort inference for survival datasets. The family of Rate Mixtures of Weibull distributions includes subject-level frailty terms as a solution to this issue. With a parametric mixing distribution assigned to the frailties, this family generates flexible hazard functions. Covariates are introduced via an Accelerated Failure Time specification for which the interpretation of the regression coefficients does not depend on the choice of mixing distribution. A weakly informative prior is proposed by combining the structure of the Jeffreys prior with a proper prior on some model parameters. This improper prior is shown to lead to a proper posterior distribution under easily satisfied conditions. By eliciting the proper component of the prior through the coefficient of variation of the survival times, prior information is matched for different mixing distributions. Posterior inference on subject-level frailty terms is exploited as a tool for outlier detection. Finally, the proposed methodology is illustrated using two real datasets, one concerning bone marrow transplants and another on cerebral palsy.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">VALLEJOS201773</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{EconStat}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Incorporating unobserved heterogeneity in Weibull survival models: A Bayesian approach}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Econometrics and Statistics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{73-88}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2452-3062}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.ecosta.2017.01.005}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.sciencedirect.com/science/article/pii/S2452306217300072}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A. and Steel, Mark F.J.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Survival analysis, Frailty model, Robust modelling, Outlier detection, Posterior existence}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Science</abbr></div> <div id="doi10.1126/science.aah4115" class="col-sm-8"> <div class="title">Aging increases cell-to-cell transcriptional variability upon immune stimulation</div> <div class="author">Celia Pilar Martinez-Jimenez, Nils Eling, Hung-Chang Chen,  <em>Catalina A. Vallejos</em>, Aleksandra A. Kolodziejczyk, Frances Connor, Lovorka Stojic, Timothy F. Rayner, Michael J. T. Stubbington, Sarah A. Teichmann, Maike Roche, <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C. Marioni</a>, and Duncan T. Odom </div> <div class="periodical"> <em>Science</em> Dec 2017 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.science.org/doi/abs/10.1126/science.aah4115" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell sequencing of mouse immune cells reveals how aging destabilizes a conserved transcriptional activation program. How and why the immune system becomes less effective with age are not well understood. Martinez-Jimenez et al. performed single-cell sequencing of CD4+ T cells in old and young mice of two species. In young mice, the gene expression program of early immune activation was tightly regulated and conserved between species. However, as mice aged, the expression of genes involved in pathways responding to immune cell stimulation was not as robust and exhibited increased cell-to-cell variability. Science, this issue p. 1433 Aging is characterized by progressive loss of physiological and cellular functions, but the molecular basis of this decline remains unclear. We explored how aging affects transcriptional dynamics using single-cell RNA sequencing of unstimulated and stimulated naïve and effector memory CD4+ T cells from young and old mice from two divergent species. In young animals, immunological activation drives a conserved transcriptomic switch, resulting in tightly controlled gene expression characterized by a strong up-regulation of a core activation program, coupled with a decrease in cell-to-cell variability. Aging perturbed the activation of this core program and increased expression heterogeneity across populations of cells in both species. These discoveries suggest that increased cell-to-cell transcriptional variability will be a hallmark feature of aging across most, if not all, mammalian tissues.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">doi10.1126/science.aah4115</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Martinez-Jimenez, Celia Pilar and Eling, Nils and Chen, Hung-Chang and Vallejos, Catalina A. and Kolodziejczyk, Aleksandra A. and Connor, Frances and Stojic, Lovorka and Rayner, Timothy F. and Stubbington, Michael J. T. and Teichmann, Sarah A. and de la Roche, Maike and Marioni, John C. and Odom, Duncan T.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Aging increases cell-to-cell transcriptional variability upon immune stimulation}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{355}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6332}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1433-1436}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.aah4115}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.science.org/doi/abs/10.1126/science.aah4115}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.science.org/doi/pdf/10.1126/science.aah4115}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2016</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">RSS A</abbr></div> <div id="Vallejos_2016" class="col-sm-8"> <div class="title">Bayesian survival modelling of university outcomes</div> <div class="author"> <em>Catalina A. Vallejos</em>, and <a href="https://warwick.ac.uk/fac/sci/statistics/staff/academic-research/steel/steel_homepage" target="_blank" rel="noopener noreferrer">Mark F. J. Steel</a> </div> <div class="periodical"> <em>Journal of the Royal Statistical Society: Series A (Statistics in Society)</em> Jul 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1111/rssa.12211" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p> Dropouts and delayed graduations are critical issues in higher education systems world wide. A key task in this context is to identify risk factors associated with these events, providing potential targets for mitigating policies. For this, we employ a discrete time competing risks survival model, dealing simultaneously with university outcomes and its associated temporal component. We define survival times as the duration of the student’s enrolment at university and possible outcomes as graduation or two types of dropout (voluntary and involuntary), exploring the information recorded at admission time (e.g. educational level of the parents) as potential predictors. Although similar strategies have been previously implemented, we extend the previous methods by handling covariate selection within a Bayesian variable selection framework, where model uncertainty is formally addressed through Bayesian model averaging. Our methodology is general; however, here we focus on undergraduate students enrolled in three selected degree programmes of the Pontificia Universidad Católica de Chile during the period 2000–2011. Our analysis reveals interesting insights, highlighting the main covariates that influence students’ risk of dropout and delayed graduation.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos_2016</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{RSS A}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A. and Steel, Mark F. J.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of the Royal Statistical Society: Series A (Statistics in Society)}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Bayesian survival modelling of university outcomes}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{613--631}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{180}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/rssa.12211}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1111/rssa.12211}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">GenBio</abbr></div> <div id="Vallejos2018" class="col-sm-8"> <div class="title">Beyond comparisons of means: understanding changes in gene expression at the single-cell level</div> <div class="author"> <em>Catalina A. Vallejos</em>, Sylvia Richardson, and <a href="https://www.ebi.ac.uk/research-beta/marioni/" target="_blank" rel="noopener noreferrer">John C. Marioni</a> </div> <div class="periodical"> <em>Genome Biology</em> Jul 2016 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1186/s13059-016-0930-3" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Traditional differential expression tools are limited to detecting changes in overall expression, and fail to uncover the rich information provided by single-cell level data sets. We present a Bayesian hierarchical model that builds upon BASiCS to study changes that lie beyond comparisons of means, incorporating built-in normalization and quantifying technical artifacts by borrowing information from spike-in genes. Using a probabilistic approach, we highlight genes undergoing changes in cell-to-cell heterogeneity but whose overall expression remains unchanged. Control experiments validate our method’s performance and a case study suggests that novel biological insights can be revealed. Our method is implemented in R and available at https://github.com/catavallejos/BASiCS.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2018</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GenBio}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A. and Richardson, Sylvia and Marioni, John C.}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2016/04/15}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-016-0930-3}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Vallejos2016}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{70}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Beyond comparisons of means: understanding changes in gene expression at the single-cell level}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-016-0930-3}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-016-0930-3}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> <h2 class="year">2015</h2> <ol class="bibliography">
<li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PLOS</abbr></div> <div id="Vallejos2015" class="col-sm-8"> <div class="title">BASiCS: Bayesian Analysis of Single-Cell Sequencing Data</div> <div class="author">Vallejos Catalina A, Marioni John C, and Richardson Sylvia </div> <div class="periodical"> <em>PLOS Computational Biology</em> Jun 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1371/journal.pcbi.1004333" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Single-cell mRNA sequencing can uncover novel cell-to-cell heterogeneity in gene expression levels in seemingly homogeneous populations of cells. However, these experiments are prone to high levels of unexplained technical noise, creating new challenges for identifying genes that show genuine heterogeneous expression within the population of cells under study. BASiCS (Bayesian Analysis of Single-Cell Sequencing data) is an integrated Bayesian hierarchical model where: (i) cell-specific normalisation constants are estimated as part of the model parameters, (ii) technical variability is quantified based on spike-in genes that are artificially introduced to each analysed cell’s lysate and (iii) the total variability of the expression counts is decomposed into technical and biological components. BASiCS also provides an intuitive detection criterion for highly (or lowly) variable genes within the population of cells under study. This is formalised by means of tail posterior probabilities associated to high (or low) biological cell-to-cell variance contributions, quantities that can be easily interpreted by users. We demonstrate our method using gene expression measurements from mouse Embryonic Stem Cells. Cross-validation and meaningful enrichment of gene ontology categories within genes classified as highly (or lowly) variable supports the efficacy of our approach.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2015</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{PLOS}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1004333}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{A, Vallejos Catalina and C, Marioni John and Sylvia, Richardson}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Computational Biology}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{BASiCS: Bayesian Analysis of Single-Cell Sequencing Data}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jun</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1371/journal.pcbi.1004333}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-18}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JASS</abbr></div> <div id="Vallejos2016" class="col-sm-8"> <div class="title">Objective Bayesian Survival Analysis Using Shape Mixtures of Log-Normal Distributions</div> <div class="author"> <em>Catalina A Vallejos</em>, and Mark FJ Steel </div> <div class="periodical"> <em>Journal of the American Statistical Association</em> Jun 2015 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1080/01621459.2014.923316" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">URL</a> </div> <div class="abstract secret"> <p>Survival models such as the Weibull or log-normal lead to inference that is not robust to the presence of outliers. They also assume that all heterogeneity between individuals can be modeled through covariates. This article considers the use of infinite mixtures of lifetime distributions as a solution for these two issues. This can be interpreted as the introduction of a random effect in the survival distribution. We introduce the family of shape mixtures of log-normal distributions, which covers a wide range of density and hazard functions. Bayesian inference under nonsubjective priors based on the Jeffreys’ rule is examined and conditions for posterior propriety are established. The existence of the posterior distribution on the basis of a sample of point observations is not always guaranteed and a solution through set observations is implemented. In addition, we propose a method for outlier detection based on the mixture structure. A simulation study illustrates the performance of our methods under different scenarios and an application to a real dataset is provided. Supplementary materials for the article, which include R code, are available online.</p> </div> <div class="bibtex secret"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Vallejos2016</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{JASS}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vallejos, Catalina A and Steel, Mark FJ}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Objective Bayesian Survival Analysis Using Shape Mixtures of Log-Normal Distributions}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of the American Statistical Association}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{110}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{510}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{697-710}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Taylor &amp; Francis}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1080/01621459.2014.923316}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1080/01621459.2014.923316}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1080/01621459.2014.923316}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li>
</ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0" style="text-align: right;"> © Copyright 2023 </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script src="/assets/js/zoom.js"></script> <script src="/assets/js/common.js"></script> <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </div> </body> </html>
